
<div class="grid">
    <div class="col twelve_col nomargin shadow">
        <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
        <div class="sysmessages">
            <noscript>
                <p class="nojs">
                    <strong>Warning:</strong>
                    The NCBI web site requires JavaScript to function.
                    <a title="Learn how to enable JavaScript" target="_blank">more...</a>
                </p>
            </noscript>
        </div>
        <!--/.sysmessage-->
        <div class="wrap">
            <div class="page">
                <div class="top">
                    <div class="universal_header" id="universal_header">
                        <ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent">
                            <li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt">
                                <a class="ui-ncbimenu-link-first" role="banner" title="NCBI Home" id="ncbihome" accesskey="1">
                                    <span class="offscreen_noflow cyxy-trs-source">NCBI</span>
                                    <img src="" class="ncbi_logo" title="NCBI" alt="NCBI Logo">
                                </a>
                            </li>
                            <li class="offscreen_noflow ui-ncbimenu-item-skip access">
                                <a title="Skip to the content" tabindex="0" accesskey="3" class="cyxy-trs-source">
                                    Skip to main
                                    content<font class="cyxy-trs-target"> 跳转到主要内容</font>
                                </a>
                            </li>
                            <li class="offscreen_noflow ui-ncbimenu-item-skip access">
                                <a title="Skip to the navigation" tabindex="0" accesskey="4" class="cyxy-trs-source">
                                    Skip to
                                    navigation<font class="cyxy-trs-target"> 跳转到导航栏</font>
                                </a>
                            </li>
                            <li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset">
                                <a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown cyxy-trs-source">Resources<font class="cyxy-trs-target"> 资源</font></a>
                            </li>
                            <li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first">
                                <a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown cyxy-trs-source">How To<font class="cyxy-trs-target"> 怎么做</font></a>
                            </li>
                            <li class="offscreen_noflow ui-ncbimenu-item-skip access">
                                <a title="About My NCBI Accesskeys" tabindex="0" accesskey="0" class="cyxy-trs-source">About NCBI Accesskeys<font class="cyxy-trs-target"> 关于 NCBI Accesskeys</font></a>
                            </li>
                        </ul>
                        <div class="myncbi">
                            <span id="myncbiusername" style="display:none">
                                <a id="mnu" title="Edit account settings"></a>
                            </span>
                            <a accesskey="2" id="myncbi" style="display:none" class="cyxy-trs-source">My NCBI<font class="cyxy-trs-target"> 我的 NCBI</font></a>
                            <a id="sign_in" class="cyxy-trs-source">Sign in to NCBI<font class="cyxy-trs-target"> 登录 NCBI</font></a>
                            <a id="sign_out" style="display:none" class="cyxy-trs-source">Sign Out<font class="cyxy-trs-target"> 登出</font></a>
                        </div>
                    </div>
                    <div class="header">
                        <div class="res_logo">
                            <h1 class="img_logo">
                                <a class="pmc_logo offscreen cyxy-trs-source">PMC<font class="cyxy-trs-target"> 普通管理委员会</font></a>
                            </h1>
                            <div class="NLMLogo">
                                <a title="US National Library of Medicine" class="cyxy-trs-source">US National Library of Medicine<font class="cyxy-trs-target"> 美国国家医学图书馆</font></a>
                                <br>
                                <a title="National Institutes of Health" class="cyxy-trs-source">National Institutes of Health<font class="cyxy-trs-target"> 国立卫生研究院</font></a>
                            </div>
                        </div>
                        <div class="search">
                            <form method="get" action="/pmc/">
                                <div class="search_form">
                                    <label for="database" class="offscreen_noflow cyxy-trs-source">Search database<font class="cyxy-trs-target" contenteditable="true"> 搜索数据库</font></label>
                                    <select id="database">
                                        <optgroup label="Recent">
                                            <option value="pmc" selected="selected" class="last cyxy-trs-source" data-ac_dict="pmc-search-autocomplete">PMC<font class="cyxy-trs-target" contenteditable="true"> 普通管理委员会</font></option>
                                        </optgroup>
                                        <optgroup label="All">
                                            <option value="gquery" class="cyxy-trs-source">All Databases<font class="cyxy-trs-target" contenteditable="true"> 所有资料库</font></option>
                                            <option value="assembly" class="cyxy-trs-source">Assembly<font class="cyxy-trs-target"> 大会</font></option>
                                            <option value="biocollections" class="cyxy-trs-source">Biocollections<font class="cyxy-trs-target"> 生物收集</font></option>
                                            <option value="bioproject" class="cyxy-trs-source">BioProject</option>
                                            <option value="biosample" class="cyxy-trs-source">BioSample<font class="cyxy-trs-target"> 生物样本</font></option>
                                            <option value="biosystems" class="cyxy-trs-source">BioSystems</option>
                                            <option value="books" class="cyxy-trs-source">Books<font class="cyxy-trs-target"> 书籍</font></option>
                                            <option value="clinvar" class="cyxy-trs-source">ClinVar<font class="cyxy-trs-target"> 克林瓦尔</font></option>
                                            <option value="cdd" class="cyxy-trs-source">Conserved Domains<font class="cyxy-trs-target"> 保守域</font></option>
                                            <option value="gap" class="cyxy-trs-source">dbGaP</option>
                                            <option value="dbvar" class="cyxy-trs-source">dbVar</option>
                                            <option value="gene" class="cyxy-trs-source">Gene<font class="cyxy-trs-target"> 基因</font></option>
                                            <option value="genome" class="cyxy-trs-source">Genome<font class="cyxy-trs-target"> 基因组</font></option>
                                            <option value="gds" class="cyxy-trs-source">GEO DataSets<font class="cyxy-trs-target" contenteditable="true"> 地球同步轨道数据集</font></option>
                                            <option value="geoprofiles" class="cyxy-trs-source">GEO Profiles<font class="cyxy-trs-target" contenteditable="true"> 土力工程处简介</font></option>
                                            <option value="gtr" class="cyxy-trs-source">GTR</option>
                                            <option value="homologene" class="cyxy-trs-source">HomoloGene<font class="cyxy-trs-target"> 同源基因</font></option>
                                            <option value="ipg" class="cyxy-trs-source">Identical Protein Groups<font class="cyxy-trs-target" contenteditable="true"> 相同的蛋白质组</font></option>
                                            <option value="medgen" class="cyxy-trs-source">MedGen</option>
                                            <option value="mesh" class="cyxy-trs-source">MeSH</option>
                                            <option value="ncbisearch" class="cyxy-trs-source">NCBI Web Site<font class="cyxy-trs-target"> 网站</font></option>
                                            <option value="nlmcatalog" class="cyxy-trs-source">NLM Catalog<font class="cyxy-trs-target" contenteditable="true"> NLM 目录</font></option>
                                            <option value="nuccore" class="cyxy-trs-source">Nucleotide<font class="cyxy-trs-target"> 核苷酸</font></option>
                                            <option value="omim" class="cyxy-trs-source">OMIM</option>
                                            <option value="pmc" data-ac_dict="pmc-search-autocomplete" class="cyxy-trs-source">PMC<font class="cyxy-trs-target" contenteditable="true"> 普通管理委员会</font></option>
                                            <option value="popset" class="cyxy-trs-source">PopSet</option>
                                            <option value="protein" class="cyxy-trs-source">Protein<font class="cyxy-trs-target"> 蛋白质</font></option>
                                            <option value="proteinclusters" class="cyxy-trs-source">Protein Clusters<font class="cyxy-trs-target"> 蛋白质簇</font></option>
                                            <option value="pcassay" class="cyxy-trs-source">PubChem BioAssay</option>
                                            <option value="pccompound" class="cyxy-trs-source">PubChem Compound<font class="cyxy-trs-target" contenteditable="true"> 上海浦化化工有限公司</font></option>
                                            <option value="pcsubstance" class="cyxy-trs-source">PubChem Substance</option>
                                            <option value="pubmed" class="cyxy-trs-source">PubMed</option>
                                            <option value="snp" class="cyxy-trs-source">SNP<font class="cyxy-trs-target" contenteditable="true"> 单核苷酸多态性</font></option>
                                            <option value="sparcle" class="cyxy-trs-source">Sparcle<font class="cyxy-trs-target"> 火星</font></option>
                                            <option value="sra" class="cyxy-trs-source">SRA<font class="cyxy-trs-target" contenteditable="true"> 自动调节系统</font></option>
                                            <option value="structure" class="cyxy-trs-source">Structure<font class="cyxy-trs-target"> 结构</font></option>
                                            <option value="taxonomy" class="cyxy-trs-source">Taxonomy<font class="cyxy-trs-target"> 分类学</font></option>
                                            <option value="toolkit" class="cyxy-trs-source">ToolKit<font class="cyxy-trs-target"> 工具包</font></option>
                                            <option value="toolkitall" class="cyxy-trs-source">ToolKitAll</option>
                                            <option value="toolkitbookgh" class="cyxy-trs-source">ToolKitBookgh<font class="cyxy-trs-target" contenteditable="true"> 图尔基特/bookgh</font></option>
                                        </optgroup>
                                    </select>
                                    <div class="nowrap">
                                        <label for="term" class="offscreen_noflow cyxy-trs-source" accesskey="/">Search term<font class="cyxy-trs-target"> 搜索词</font></label>
                                        <div class="nowrap">
                                            <input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'">
                                        </div>
                                        <button id="search" type="submit" class="button_search nowrap cyxy-trs-source" cmd="go">Search<font class="cyxy-trs-target"> 搜寻</font></button>
                                    </div>
                                </div>
                            </form>
                            <ul class="searchlinks inline_list">
                                <li>
                                    <a class="cyxy-trs-source">Advanced<font class="cyxy-trs-target"> 高级</font></a>
                                </li>
                                <li>
                                    <a class="cyxy-trs-source">Journal list<font class="cyxy-trs-target"> 日记清单</font></a>
                                </li>
                                <li class="help">
                                    <a target="_blank" class="cyxy-trs-source">Help<font class="cyxy-trs-target"> 救命</font></a>
                                </li>
                            </ul>
                        </div>
                    </div>



                    <!--<component id="Page" label="headcontent"/>-->

                </div>
                <div class="content">
                    <!-- site messages -->
                    <div class="container">
                        <div id="maincontent" class="content eight_col col">
                            <div class="navlink-box">
                                <ul class="page-breadcrumbs inline_list small">
                                    <li class="journal-list">
                                        <a class="navlink cyxy-trs-source">Journal List<font class="cyxy-trs-target"> 日记清单</font></a>
                                    </li>
                                    <li class="archive">
                                        <a class="navlink cyxy-trs-source">Nucleic Acids Res<font class="cyxy-trs-target"> 核酸</font></a>
                                    </li>
                                    <li class="issue-page">
                                        <a class="navlink cyxy-trs-source">v.43(15); 2015 Sep 3<font class="cyxy-trs-target"> V. 43(15) ; 2015 Sep 3</font></a>
                                    </li>
                                    <li class="accid cyxy-trs-source">PMC4551934</li>
                                </ul>
                            </div>

                            <!-- Journal banner -->
                            <div class="pmc-page-banner whole_rhythm">
                                <div>
                                    <img src="" alt="Logo of nar" usemap="#logo-imagemap">
                                    <map id="logo-imagemap" name="logo-imagemap">
                                        <area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" target="pmc_ext" ref="reftype=publisher&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A">
                                    </map>
                                </div>
                            </div>

                            <!--component id='MainPortlet' label='search-reference'/-->
                            <!-- Book content -->
                            <div class="">



                                <div class="hide-overflow article lit-style content pmc-wm slang-all page-box">
                                    <!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'">
                                        <div class="fm-sec half_rhythm no_top_margin">
                                            <div class="fm-citation half_rhythm no_top_margin clearfix">
                                                <div class="inline_block eight_col va_top">
                                                    <div>
                                                        <div>
                                                            <span class="cit cyxy-trs-source"><span id="pmcmata" class="cyxy-trs-source">Nucleic Acids Res<font class="cyxy-trs-target"> 核酸</font></span>. 2015 Sep 3; 43(15): 7544–7555. <font class="cyxy-trs-target" contenteditable="true"> . 2015 Sep 3; 43(15) : 7544-7555</font></span>
                                                        </div>
                                                        <div>
                                                            <span class="fm-vol-iss-date cyxy-trs-source">Published online 2015 Jul 15. <font class="cyxy-trs-target" contenteditable="true"> 发表于2015年7月15日</font></span>
                                                            <span class="doi cyxy-trs-source">
                                                                doi:&nbsp;<font class="cyxy-trs-target"> 干杯:</font><a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI" class="cyxy-trs-source">10.1093/nar/gkv702<font class="cyxy-trs-target"> 10.1093/nar/gv702</font></a>
                                                            </span>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="inline_block four_col va_top show-overflow align_right">
                                                    <div class="fm-citation-ids">
                                                        <div class="fm-citation-pmcid">
                                                            <span class="fm-citation-ids-label cyxy-trs-source">PMCID: </span>
                                                            <span class="cyxy-trs-source">PMC4551934</span>
                                                        </div>
                                                        <div class="fm-citation-pmid cyxy-trs-source">
                                                            PMID: <a>26184872</a>
                                                        </div>
                                                    </div>
                                                </div>
                                            </div>
                                            <h1 class="content-title cyxy-trs-source cyxy-trs-source-ted"><mark id="mark-id-1" class="tagged tagged-node" tagtype="ts" c-id="1" c-type="T" c-name="AptamerType" style="background-color: #FFFFCC;">{</mark>RNA<mark id="mark-id-1" class="tagged tagged-node" tagtype="te" c-id="1" c-type="T" c-name="AptamerType" style="background-color: #FFFFCC;">}</mark> aptamer inhibitors of a restriction endonuclease</h1><h1 class="content-title cyxy-trs-source cyxy-trs-target" contenteditable="true">限制性内切酶的核酸适配子抑制剂</h1>
                                            <div class="half_rhythm">
                                                <div class="contrib-group fm-author cyxy-trs-source">
                                                    <a class="affpopup cyxy-trs-source" co-rid="_co_idm140607998701440" co-class="co-affbox">Estefanía Mondragón<font class="cyxy-trs-target"> 埃斯特凡尼亚 · 蒙德拉贡</font></a>
                                                    <sup>1</sup> and  <font class="cyxy-trs-target"> 及</font><a class="affpopup cyxy-trs-source" co-rid="_co_idm140607991882464" co-class="co-affbox">L. James Maher, III<font class="cyxy-trs-target"> 詹姆斯 · 马赫三世</font></a>
                                                    <sup class="cyxy-trs-source">2,<font class="cyxy-trs-target"> 二、,</font></sup>
                                                    <sup>*</sup>
                                                </div>
                                                <div style="display:none" class="contrib-group aff-tip">
                                                    <div id="_co_idm140607998701440">
                                                        <h3 class="no_margin cyxy-trs-source cyxy-trs-source-ted">Estefanía Mondragón</h3><h3 class="no_margin cyxy-trs-source cyxy-trs-target" contenteditable="true">埃斯特凡尼亚 · 蒙德拉贡</h3>
                                                        <p class="cyxy-trs-source cyxy-trs-source-ted"><sup>1</sup>Neurobiology of Disease track, Mayo Graduate School, 200 First St SW, Rochester, MN 55905, USA</p><p class="cyxy-trs-source cyxy-trs-target" contenteditable="true">1Neurobiology of Disease track，Mayo Graduate School，200 First St SW，Rochester，MN 55905，USA</p>
                                                        <div class="cyxy-trs-source">
                                                            Find articles by <font class="cyxy-trs-target"> 查找文章</font><a class="cyxy-trs-source">Estefanía Mondragón<font class="cyxy-trs-target"> 埃斯特凡尼亚 · 蒙德拉贡</font></a>
                                                        </div>
                                                    </div>
                                                    <div id="_co_idm140607991882464">
                                                        <h3 class="no_margin cyxy-trs-source cyxy-trs-source-ted">L. James Maher, III</h3><h3 class="no_margin cyxy-trs-source cyxy-trs-target" contenteditable="true">詹姆斯 · 马赫三世</h3>
                                                        <p class="cyxy-trs-source cyxy-trs-source-ted"><sup>2</sup>Department of Biochemistry and Molecular Biology, Mayo Clinic College of Medicine, 200 First St SW, Rochester, MN 55905, USA</p><p class="cyxy-trs-source cyxy-trs-target" contenteditable="true">2Department of biomization and Molecular Biology，Mayo Clinic College of Medicine，200 First St SW，Rochester，MN 55905，USA</p>
                                                        <div class="cyxy-trs-source">
                                                            Find articles by <font class="cyxy-trs-target"> 查找文章</font><a class="cyxy-trs-source">L. James Maher, III<font class="cyxy-trs-target"> 詹姆斯 · 马赫三世</font></a>
                                                        </div>
                                                    </div>
                                                </div>
                                            </div>
                                            <div class="fm-panel half_rhythm">
                                                <div class="togglers">
                                                    <a class="pmctoggle cyxy-trs-source" rid="idm140607996797376_ai">Author information<font class="cyxy-trs-target"> 作者信息</font></a>
                                                    <a class="pmctoggle cyxy-trs-source" rid="idm140607996797376_an">Article notes<font class="cyxy-trs-target"> 文章注释</font></a>
                                                    <a class="pmctoggle cyxy-trs-source" rid="idm140607996797376_cpl">Copyright and License information<font class="cyxy-trs-target"> 版权和许可证信息</font></a>
                                                    <a class="cyxy-trs-source">Disclaimer<font class="cyxy-trs-target"> 免责声明</font></a>
                                                </div>
                                                <div class="fm-authors-info fm-panel hide half_rhythm" id="idm140607996797376_ai" style="display:none">
                                                    <div class="fm-affl cyxy-trs-source" lang="en" id="AFF1"><sup>1</sup>Neurobiology of Disease track, Mayo Graduate School, 200 First St SW, Rochester, MN 55905, USA<font class="cyxy-trs-target" contenteditable="true"> 疾病的神经生物学轨道，梅奥研究生院，200第一圣西南，罗切斯特，明尼苏达州55905，美国</font></div>
                                                    <div class="fm-affl cyxy-trs-source" lang="en" id="AFF2"><sup>2</sup>Department of Biochemistry and Molecular Biology, Mayo Clinic College of Medicine, 200 First St SW, Rochester, MN 55905, USA<font class="cyxy-trs-target" contenteditable="true"> 生物化学与分子生物学系，梅奥诊所医学院，200第一圣西南，罗切斯特，MN 55905，美国</font></div>
                                                    <div id="COR1" class="cyxy-trs-source">
                                                        <sup>*</sup>To whom correspondence should be addressed. Tel: +1 507 284 9041; Fax: +1 507 284 2053; Email: <font class="cyxy-trs-target" contenteditable="true"> 电话: + 15072849041; 传真: + 15072842053; 电邮:</font><a data-email="ude.oyam@reham" class="oemail">ude.oyam@reham</a>
                                                    </div>
                                                </div>
                                                <div class="fm-article-notes fm-panel hide half_rhythm" id="idm140607996797376_an" style="display:none">
                                                    <div class="fm-pubdate half_rhythm cyxy-trs-source">Received 2015 Apr 16; Revised 2015 Jun 25; Accepted 2015 Jun 29.<font class="cyxy-trs-target" contenteditable="true"> 2015 Apr 16; Revised 2015 Jun 25; Accepted 2015 Jun 29</font></div>
                                                </div>
                                                <div class="permissions fm-panel half_rhythm hide" id="idm140607996797376_cpl" style="display:none">
                                                    <div class="fm-copyright half_rhythm cyxy-trs-source"><a class="cyxy-trs-source">Copyright<font class="cyxy-trs-target"> 版权所有</font></a> © The Author(s) 2015. Published by Oxford University Press on behalf of Nucleic Acids Research.<font class="cyxy-trs-target" contenteditable="true"> 作者(s)2015。牛津大学出版社代表核酸研究出版</font></div>
                                                    <div class="license half_rhythm cyxy-trs-source cyxy-trs-source">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<font class="cyxy-trs-target" contenteditable="true"> 这是一篇开放获取的文章，根据知识共享的归属许可条款发布</font><a data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.<font class="cyxy-trs-target" contenteditable="true"> ) ，允许在任何媒介上不受限制地重复使用、分发和复制，只要原著被适当引用</font></div>
                                                </div>
                                            </div>
                                            <div id="pmclinksbox" class="links-box whole_rhythm">
                                                <div class="fm-panel">
                                                    <div class="cyxy-trs-source cyxy-trs-source">This article has been <font class="cyxy-trs-target" contenteditable="true"> 这篇文章已经被</font><a class="cyxy-trs-source">cited by<font class="cyxy-trs-target"> 引用</font></a> other articles in PMC.<font class="cyxy-trs-target" contenteditable="true"> 在 PMC 的其他文章</font></div>
                                                </div>
                                            </div>
                                        </div>
                                        <div class="sec"></div>
                                        <div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation">
                                            <h2 class="cyxy-trs-source cyxy-trs-source-ted">Associated Data</h2><h2 class="cyxy-trs-source cyxy-trs-target">相关数据</h2>
                                            <dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin">
                                                <dt>
                                                    <a rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle cyxy-trs-source">Supplementary Materials<font class="cyxy-trs-target"> 补充资料</font></a>
                                                </dt>
                                                <dd id="data-suppmats" style="display: none;">
                                                    <div class="half_rhythm">
                                                        <div class="caption half_rhythm no_bottom_margin cyxy-trs-source">
                                                            SUPPLEMENTARY DATA
                                                            <font class="cyxy-trs-target"> 补充资料</font>
                                                        </div>

                                                        <div>
                                                            <a data-ga-action="click_feat_suppl">supp_43_15_7544__index.html</a>
                                                            <span style="color:gray" class="cyxy-trs-source"> (738 bytes)<font class="cyxy-trs-target" contenteditable="true"> (738字节)</font></span>
                                                        </div>
                                                        <div class="small guid cyxy-trs-source">GUID:&nbsp;1B1F7FED-A236-4CA9-8EC9-357A69408FA6<font class="cyxy-trs-target" contenteditable="true"> GUID: 1B1F7FED-A236-4CA9-8EC9-357A69408FA6</font></div>
                                                        <div>
                                                            <a data-ga-action="click_feat_suppl">supp_gkv702_Mondragon_and_Maher_supplemental.pdf</a>
                                                            <span style="color:gray" class="cyxy-trs-source"> (1.0M)<font class="cyxy-trs-target" contenteditable="true"> (1.0 m)</font></span>
                                                        </div>
                                                        <div class="small guid cyxy-trs-source">GUID:&nbsp;CC0F2634-7073-42D0-8DA5-EC35CEBBDD8E<font class="cyxy-trs-target" contenteditable="true"> GUID: cc0f2634-7073-42d0-8da5-ec35cebdd8e</font></div>
                                                    </div>
                                                </dd>
                                            </dl>
                                        </div>
                                        <div id="idm140607998619408" lang="en" class="tsec sec">
                                            <h2 class="head no_bottom_margin cyxy-trs-source cyxy-trs-source-ted" id="idm140607998619408title" style="text-transform: uppercase;">Abstract</h2><h2 class="head no_bottom_margin cyxy-trs-source cyxy-trs-target" id="idm140607998619408title" style="text-transform: uppercase;">摘要</h2>
                                            <!--article-meta--><div>
                                                <p id="__p1" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted">
                                                    Restriction endonucleases (REases) recognize and cleave short palindromic DNA sequences, protecting bacterial cells against bacteriophage infection by attacking foreign DNA. We are interested in the potential of folded RNA to mimic DNA, a concept that might be applied to inhibition of DNA-binding proteins. As a model system, we sought RNA aptamers against the REases BamHI, PacI and KpnI using systematic evolution of ligands by exponential enrichment (<mark id="mark-id-3" class="tagged tagged-node" tagtype="ts" c-id="3" c-type="T" c-name="ScreenMethod" style="background-color: #0099CC;">{</mark>SELEX<mark id="mark-id-3" class="tagged tagged-node" tagtype="te" c-id="3" c-type="T" c-name="ScreenMethod" style="background-color: #0099CC;">}</mark>). After 20 rounds of selection under different stringent conditions, we identified the 10 most enriched RNA aptamers for each REase. Aptamers were screened for binding and specificity, and assayed for REase inhibition. We obtained eight high-affinity (<em>K</em>
                                                    <sub>d</sub> ∼<mark id="mark-id-2" class="tagged tagged-node" tagtype="ts" c-id="2" c-type="T" c-name="Affinity" style="background-color: #FFFF99;">{</mark>12-30 nM<mark id="mark-id-2" class="tagged tagged-node" tagtype="te" c-id="2" c-type="T" c-name="Affinity" style="background-color: #FFFF99;">}</mark>) selective competitive inhibitors (IC<sub>50</sub>&nbsp;∼20-150 nM) for KpnI. Predicted RNA secondary structures were confirmed by in-line attack assay and a 38-nt derivative of the best anti-KpnI aptamer was sufficient for inhibition. These competitive inhibitors presumably act as KpnI binding site analogs, but lack the primary consensus KpnI cleavage sequence and are not cleaved by KpnI, making their potential mode of DNA mimicry fascinating. Anti-REase RNA aptamers could have value in studies of REase mechanism and may give clues to a code for designing RNAs that competitively inhibit DNA binding proteins including transcription factors.
                                                </p><p id="__p1" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">限制性内切酶(REases)识别并切断短的回文 DNA 序列，通过攻击外源 DNA 保护细菌细胞免受噬菌体感染。我们感兴趣的是折叠 RNA 模拟 DNA 的潜力，这个概念可能应用于抑制 DNA 结合蛋白质。作为一个模型系统，我们使用配体指数增强系统进化技术({ SELEX })对 REases BamHI、 PacI 和 KpnI 进行 RNA 适配子的筛选。经过20轮不同条件下的筛选，筛选出每个 REase 最适合的10个 RNA 适配子。筛选适配子，检测其结合力和特异性，并测定其对 REase 的抑制作用。我们获得了8个高亲和力的 kd-12-30nm 选择性竞争抑制剂(IC50∼20-150nm)。通过在线攻击实验证实了预测的 RNA 二级结构，最佳抗 kpni 核酸适配体的38nt 衍生物足以抑制 RNA 二级结构。这些竞争性抑制剂可能作为 KpnI 结合位点的类似物，但缺乏 KpnI 裂解序列的主要共识，而且不能被 KpnI 裂解，这使得它们潜在的 DNA 模拟模式非常迷人。抗 REase RNA 适配子在研究 REase 机制方面具有重要价值，可能为设计竞争性抑制包括转录因子在内的 DNA 结合蛋白的 RNA 提供线索。</p>
                                            </div>
                                        </div>
                                        <div id="SEC1" class="tsec sec">
                                            <h2 class="head no_bottom_margin cyxy-trs-source cyxy-trs-source-ted" id="SEC1title" style="text-transform: uppercase;">INTRODUCTION</h2><h2 class="head no_bottom_margin cyxy-trs-source cyxy-trs-target" id="SEC1title" style="text-transform: uppercase;">引言</h2>
                                            <p id="__p2" class="p p-first cyxy-trs-source cyxy-trs-source-ted">DNA binding proteins have a myriad of roles in the replication, transcription, repair and degradation of DNA. DNA restriction endonucleases (REases) are prokaryotic enzymes that cleave double-stranded DNA within or near their recognition sites. REases require divalent cations (typically Mg<sup>2+</sup>) for their activity and are thought to protect the host cell from the invasion of foreign DNA, such as viral DNA, by detecting DNA methylation status (<a rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922014">1</a>). It has been argued that REases arose as part of poison-antidote systems (<a rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921988">2</a>). Type II REases have been widely studied since their discovery in the 1970s (<a rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922000">3</a>,<a rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921985">4</a>) and have become essential tools for molecular cloning.</p><p id="__p2" class="p p-first cyxy-trs-source cyxy-trs-target" contenteditable="true">DNA 结合蛋白在 DNA 的复制、转录、修复和降解过程中发挥着多种作用。DNA 限制性内切酶(REases)是将双链 DNA 在其识别位点内或附近切断的原核酶。Eases 需要二价阳离子(通常是 Mg2 +)来保持其活性，并且通过检测 DNA 甲基化状态(1)来保护宿主细胞免受外源 DNA (如病毒 DNA)的侵袭。有人认为 REases 是作为解毒系统的一部分出现的(2)。自20世纪70年代(3,4)发现以来，第二类 REases 已经被广泛研究，并且已经成为分子克隆的基本工具。</p>
                                            <p id="__p3" class="cyxy-trs-source cyxy-trs-source-ted">Our long-term aim is to design short RNA aptamers as specific inhibitors of DNA binding proteins such as transcription factors. We are approaching this challenge by first studying RNA aptamers selected from random libraries for DNA mimicry using various kinds of DNA binding protein targets. The premise that RNA aptamers might competitively inhibit DNA binding proteins is motivated by natural examples such as <em>Xenopus laevis</em> transcription factor IIIA (TFIIIA). This zinc finger protein engages the internal control region of the 5S RNA gene, as well as the 5S RNA transcript itself (<a rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921989">5</a>–<a rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922016">9</a>). The alternative binding states are mutually exclusive, such that the RNA functions as a transcription factor decoy to effect product inhibition. Other natural examples may exist, such as the transcription factor bicoid (<a rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921987">9</a>–<a rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921984">12</a>) and several other RNA decoys for DNA binding proteins have been artificially selected using SELEX (<a rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922027">13</a>,<a rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922017">14</a>) against forms of NF-κB (<a rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922020">15</a>–<a rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921999">17</a>), heat shock transcription factor (<a rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922009">18</a>), TFIIB (<a rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922019">19</a>) and RUNX1 (<a rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922007">20</a>). In the case of anti-NF-κB p50, the RNA aptamer has been studied at high resolution both free and bound to its protein target and has been shown to adopt a pre-formed tertiary structure that resembles closely the structure of DNA (<a rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921995">21</a>–<a rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921991">23</a>). Likewise, RUNX1 structure has been determined in DNA and RNA aptamer complexes. RNA is again observed to mimic DNA (<a rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922008">24</a>,<a rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922002">25</a>).</p><p id="__p3" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">我们的长期目标是设计短 RNA 适配体作为 DNA 结合蛋白(如转录因子)的特异性抑制剂。我们首先通过研究从随机文库中选取的 RNA 适配体，利用各种 DNA 结合蛋白靶标进行 DNA 模拟，来迎接这一挑战。RNA 适配子可能竞争性地抑制 DNA 结合蛋白的前提是受到自然例子的推动，如非洲爪蟾的转录因子 iiiia (TFIIIA)。这种锌指蛋白结合了5S RNA 基因的内部控制区，以及5S RNA 转录本本身(5-9)。这两种可选择的结合状态是互相排斥的，因此 RNA 的功能就像一个转录因子诱骗物来实现产物抑制。其他自然的例子可能存在，例如转录因子二元体(9-12)和其他几个 RNA 诱骗 DNA 结合蛋白已被人工选择使用 SELEX (13,14)对核因子 κb (15-17) ，热休克转录因子(18) ，TFIIB (19)和 RUNX1(20)的形式。在抗 nf-κb p50的情况下，RNA 适配子已被研究在高分辨率的自由和结合到其蛋白质目标，并已被证明采用一个预先形成的三级结构非常类似于 DNA (21-23)的结构。同样，RUNX1的结构已经在 DNA 和 RNA 适配体复合物中被确定。再次观察到 RNA 模拟 DNA (24,25)。</p>
                                            <p id="__p4" class="p p-last cyxy-trs-source cyxy-trs-source-ted">With the goal of collecting additional examples of RNA mimics of double-stranded DNA we applied SELEX to three Type II REases: BamHI, KpnI and PacI. We report the identification of several high-affinity RNA aptamers that act as selective competitive inhibitors of KpnI. These novel aptamers provide new examples for future structural analysis with the goal of decoy design principles.</p><p id="__p4" class="p p-last cyxy-trs-source cyxy-trs-target" contenteditable="true">为了收集更多的双链 DNA RNA 模拟样本，我们将 SELEX 应用于三个 II 型模拟样本: BamHI、 KpnI 和 PacI。我们鉴定了几种高亲和力的 RNA 适配子，它们是 KpnI 的选择性竞争抑制剂。这些新型适配子为进一步的结构分析提供了新的实例，目的是遵循诱饵设计原则。</p>
                                        </div>
                                        <div id="SEC2" class="tsec sec">
                                            <h2 class="head no_bottom_margin cyxy-trs-source cyxy-trs-source-ted" id="SEC2title" style="text-transform: uppercase;">MATERIALS AND METHODS</h2><h2 class="head no_bottom_margin cyxy-trs-source cyxy-trs-target" id="SEC2title" style="text-transform: uppercase;" contenteditable="true">物料及方法</h2>
                                            <div id="SEC2-1" class="sec sec-first">
                                                <h3 id="SEC2-1title" class="cyxy-trs-source cyxy-trs-source-ted">RNA libraries</h3><h3 id="SEC2-1title" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">RNA 文库</h3>
                                                <p id="__p5" class="p p-first cyxy-trs-source cyxy-trs-source-ted">
                                                    The RNA library used for the <em>in</em>
                                                    <em>vitro</em> selections against REases were based on the <em>in</em>
                                                    <em>vitro</em> and <em>in</em>
                                                    <em>vivo</em> selections previously performed in our laboratory against transcription factor NF-κB (<a rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922023">16</a>,<a rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921994">17</a>). The results of these studies using a 60-nt random library showed that the minimal active domain was an imperfect 31-nt hairpin. This hairpin, defined by boundary and mutagenesis studies, has a 7-nt loop flanked by 17 critical nucleotides that are recognized by NF-κB. Far from being an asymmetric internal loop as predicted by secondary structure prediction algorithms, these nucleotides participate in a continuous stem composed of canonical and non-canonical interactions that result in striking mimicry of the major groove of a DNA double helix (<a rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922010">17</a>). Subsequent <em>in</em>
                                                    <em>vivo</em> studies and re-selections were performed to select variants with improved activity in yeast (<a rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921986">16</a>). The resulting optimized aptamers displayed the same hairpin stem sequence but with a GUAA tetraloop replacing the original 7-nt loop. This interpretation was confirmed by subsequent high-resolution structural studies (<a rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922021">21</a>,<a rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922029">22</a>). We used this ‘scaffold’ as the basis for the present selections, reasoning that desired aptamers will mimic the structure of double-stranded DNA through variation of the theme exemplified by the anti-NF-κB aptamer. Therefore, the structured RNA libraries (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A)</span>
                                                        </span>
                                                        <span>1A</span>
                                                    </a>) used for selection were 67-nt in length with a potential tetraloop (GUAA) flanked by two random regions: 10 nt upstream and 9 nt downstream of the tetraloop. These regions were then flanked by short complementary sequences supporting the weak stem, flanked by constant regions used for manipulation of the libraries (T7 RNA polymerase promoter, reverse transcriptase priming and polymerase chain reaction (PCR) priming sequences). Thus, the structure of the α-p50 aptamer provided inspiration for design of asymmetric random regions in the context of a small hairpin.
                                                </p><p id="__p5" class="p p-first cyxy-trs-source cyxy-trs-target" contenteditable="true">用于针对 REases 的体外选择的 RNA 文库是基于我们实验室先前针对转录因子核因子 κb 的体外和体内选择。这些研究结果使用60-nt 的随机文库表明，最小活性域是一个不完全的31-nt 发夹。该发夹由边界和突变研究确定，7-nt 环由17个核苷酸组成，这些核苷酸被 NF-κB 识别。与二级结构预测算法所预测的非对称内环不同，这些核苷酸参与了由正则和非正则相互作用组成的连续的序列，这些正则和非正则相互作用引人注目地模拟了 DNA 双螺旋(17)的主槽。随后进行体内研究和重新选择，以选择在酵母中活性提高的变异体(16)。优化后的适配体具有相同的发夹茎序列，但以 GUAA 四肽取代了原来的7-nt 环。随后的高分辨率结构研究(21,22)证实了这种解释。我们使用这个“支架”作为目前选择的基础，推理所需的适配子将通过抗 nf-κb 适配子所示例的主题变异模拟双链 DNA 的结构。因此，用于筛选的结构化 RNA 库(图(图1A)1A)长度为67nt，其中一个潜在的四肽(GUAA)位于四肽的上游10nt 和下游9nt 两个随机区域。这些区域的两侧是支持弱茎的短互补序列，两侧是用于操纵库的常量区域(T7RNA聚合酶启动子、逆转录酶引发和聚合酶链式反应引发序列)。因此，α-p50适配子的结构为设计小发夹情况下的非对称随机区域提供了灵感。</p>
                                                <!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1">
                                                    <a target="figure" rid-figpopup="F1" rid-ob="ob-F1">
                                                        <!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140608003337616" class="figure">
                                                            <img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gkv702fig1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gkv702fig1.jpg" src="http://img.staging.deepbiogroup.com/AptamerImageTotals/822838600000.jpg">
                                                        </div>
                                                    </a><div id="largeobj_idm140608003337616" class="largeobj-link align_right" style="display: none">
                                                        <a target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>

                                                    <div class="icnblk_cntnt" id="lgnd_F1">
                                                        <div>
                                                            <a class="figpopup cyxy-trs-source" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.<font class="cyxy-trs-target"> 图1</font></a>
                                                        </div>
                                                        <!--caption a7--><div class="caption">
                                                            <p id="__p6" class="cyxy-trs-source cyxy-trs-source-ted">
                                                                <em>In</em>
                                                                <em>vitro</em> selection process. (<strong>A</strong>) RNA aptamer library format, random region and tetraloop highlighted in black. (<strong>B</strong>) Fraction of RNA recovered from selections against BamHI (blue circles), KpnI (green triangles) and PacI (red squares), as a function of selection round.
                                                            </p><p id="__p6" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">体外筛选过程。(a) RNA 适配体库格式、随机区域和四聚体以黑色突出显示。(b)从 BamHI (蓝色圆圈)、 KpnI (绿色三角形)和 PacI (红色方块)的选择中提取的 RNA 的分数，作为选择轮的函数。</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <p id="__p7" class="p p-last cyxy-trs-source cyxy-trs-source-ted">
                                                    DNA oligonucleotides were synthesized commercially (IDT) and purified by the manufacturer using gel filtration chromatography. The synthetic DNA template for the selection pool was 5′-TA<sub>2</sub>TACGACTCACTATAG<sub>3</sub>ATATC<sub>2</sub>TC<sub>2</sub>ATGATN<sub>10</sub>G-TA<sub>2</sub>N<sub>9</sub>ATCATGCATGA<sub>2</sub>GCGT<sub>2</sub>C<sub>2</sub>ATAT<sub>5</sub> where N indicates any base. Nucleotides in the random region were synthesized from a mixture of phosphoramidites adjusted for the relative coupling efficiency of each monomer. The library template was amplified by PCR using primers LJM-4485 5′-TA<sub>2</sub>TACGACTCACTATAG<sub>3</sub>ATATC<sub>2</sub> and LJM-4486 5′A<sub>5</sub>TATG<sub>2</sub>A<sub>2</sub>CGCT<sub>2</sub>CATGCAT. Radiolabeled RNA aptamers were prepared by <em>in</em>
                                                    <em>vitro</em> transcription of double-stranded DNA templates using the Ampliscribe T7 High Yield Transcription Kit (Epicentre) as described by the manufacturer, where standard 10-μl reactions included 2 μl [α-<sup>32</sup>P]-CTP (800Ci/mmol, 10 mCi/ml).
                                                </p><p id="__p7" class="p p-last cyxy-trs-source cyxy-trs-target" contenteditable="true">DNA 寡核苷酸的合成(IDT)和纯化的制造商使用凝胶过滤色谱。选择池的合成 DNA 模板为5′-ta2tacgactcactatag3atatc22atgatc22atgatn10g-ta2n9atcatga2gcgt2c2at5，其中 n 表示任意碱基。以磷酰胺混合物为原料，通过调整各单体的相对偶联效率，合成了随机区域的核苷酸。利用引物 ljm-44855′-TA2TACGACTCACTATAG3ATATC2和 ljm-44865′ A5TATG2A2CGCT2CATGCAT 扩增文库模板。利用生产厂家介绍的 amplescribe T7高产转录试剂盒(Epicentre)对双链 DNA 模板进行体外转录，制备了放射标记的 RNA 适配子，其中标准的10μl 反应包括2μl [ α-32p ]-CTP (800Ci/mmol，10mci/ml)。</p>
                                            </div>
                                            <div id="SEC2-2" class="sec">
                                                <h3 id="SEC2-2title" class="cyxy-trs-source cyxy-trs-source-ted">Protein expression and purification</h3><h3 id="SEC2-2title" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">蛋白质的表达和纯化</h3>
                                                <p id="__p8" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted">
                                                    Commercial REases and purified preparations of BamHI (E111A mutant), KpnI and PacI for <em>in</em>
                                                    <em>vitro</em> selection and binding assays were obtained from New England Biolabs.
                                                </p><p id="__p8" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">从 New England Biolabs 获得了适用于体外筛选和结合分析的商品化酶和纯化酶制剂，包括 E111A 突变体、 KpnI 和 PacI。</p>
                                            </div>
                                            <div id="SEC2-3" class="sec">
                                                <h3 id="SEC2-3title" class="cyxy-trs-source cyxy-trs-source-ted">SELEX protocol</h3><h3 id="SEC2-3title" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">SELEX 协议</h3>
                                                <p id="__p9" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted">
                                                    Naïve libraries for selection were prepared by PCR amplification of 5 pmol double-stranded DNA template. Because each template contains 19 random positions, the theoretical complexity of this library is 3 × 10<sup>11</sup> unique sequences. A 5 pmol sample contains about 10 copies of each unique sequence. PCR reactions (100 μl) contained 20 mM Tris pH 8.4, 50 mM KCl, 4 mM MgCl<sub>2</sub>, 200 nM dNTPs, 400 nM of each primer, 20 nM DNA template and 0.1 U/μl recombinant Taq polymerase (Invitrogen). The optimum number of PCR cycles was determined by qualitative appearance of the amplification product on an 8% polyacrylamide gel (29:1 acrylamide:bisacrylamide), electrophoresed in 0.5× Tris-borate EDTA (TBE)&nbsp;buffer and stained with ethidium bromide. PCR was achieved by 2 min at 95°C as initial denaturation, followed by 3 cycles of 30 s at 95°C for denaturation, 30 s at 52°C as annealing, 30 s at 72°C for extension, followed by 5 cycles of 30 s at 95°C for denaturation, 30 s at 65°C as annealing, 30 s at 72°C for extension and final extension for 5 min at 72°C. Products were separated by electrophoresis through 2.5% agarose gels in 1× TAE buffer and the major product purified using a QIAquick gel extraction kit (Qiagen). Initial RNA pools were transcribed from 6 pmol of DNA template (representing ∼10<sup>12</sup> sequences) using an Ampliscribe T7 Transcription Kit (Epicentre) as described above. RNA was purified by denaturing 10% polyacrylamide gel electrophoresis (29:1 acrylamide:bisacrylamide, 8 M urea, 0.5X TBE), visualized by UV shadowing, excised and eluted from gel pieces overnight at room temperature with agitation in 2× PK buffer (200 mM Tris pH 7.6, 2.5 mM ethylenediaminetetraacetic acid (EDTA), 300 mM NaCl and 2% sodium dodecyl sulphate). The eluted material was extracted with an equal volume of phenol/chloroform (1:1, v/v) and RNA was precipitated by the addition of 2.5 volumes of ethanol. For initial selections, 23 pmol of pool RNA representing ∼43 copies each of 3 × 10<sup>11</sup> DNA templates was used as a 0.46 nM solution in a volume of 50 ml in the presence of 2 nM target protein in the appropriate selection buffer (SB) such that REases are not catalytically active: BamHI(E111A): 10 mM HEPES pH 7.5, 400 mM NaCl, 1 mM MgCl<sub>2</sub>; KpnI: 10 mM HEPES pH 7.5, 100 mM NaCl, 1 mM CaCl<sub>2</sub>; and PacI: 10 mM HEPES pH 7.5, 10 mM NaCl, 1 mM CaCl<sub>2.</sub> Prior to selection all RNA pools were incubated in SB at 70°C for 5 min and allowed to cool to room temperature to promote RNA folding. Selections were performed by incubation of RNAs for 45 min in SB buffer with or without protein followed by vacuum partitioning using nitrocellulose filtration (Millipore HAWP02500, 0.45 μm HA filters). Unbound RNAs were washed from the filter using two aliquots of 10 ml SB. Captured RNA was then quantitated by Cerenkov radiation detected on filters using the <sup>3</sup>H setting of a scintillation counter. Bound RNA was eluted in 500 μl 2× PK buffer at 65°C for 30 min. The eluted solution was extracted with an equal volume of phenol:chloroform (1:1) and RNA precipitated from ethanol using 40 μg of glycogen as carrier. Before round 3 and subsequent odd rounds negative selections were performed by exposing the RNA library to nitrocellulose filter strips and processing the supernatant for selection. To prepare cDNA, one-third of the recovered RNA was reverse transcribed using primer LJM-4486 (5′-A<sub>5</sub>TATG<sub>2</sub>A<sub>2</sub>CGCT<sub>2</sub>CATGCAT). Prior to the addition of MMLV reverse transcriptase and dNTPs, the primer was annealed by incubating for 5 min at 65°C and 5 min at room temperature. Each 40-μl reverse transcription reaction contained 1.25 μM primer LJM-4486, 400 μM dNTPs, 50 mM KCl, 6 mM MgCl<sub>2</sub>, 50 mM Tris pH 8.0 and 2 μl 200 U/μl MMLV reverse transcriptase (Invitrogen.) Reactions were incubated at room temperature for 10 min and at 37°C for 30 min. One-half of the each reaction was used for optimization PCR. After PCR amplification of the remaining cDNA, DNA libraries were purified using a PCR Purification Kit (Qiagen). To prepare RNA pools for subsequent rounds of selection, 200 ng (4 pmol) purified DNA was transcribed as previously described. To monitor binding and recovery of radiolabeled RNAs at each round of selection, nitrocellulose filter radioactivity was compared to a fraction of total radioactivity to determine the fraction of RNA bound to the protein target. Twenty rounds of selection were performed for each enzyme with maximum RNA recovery for BamHI reached at cycle 14, for KpnI at cycle 15 and for PacI at cycle 19&nbsp;(Figure&nbsp;<a target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1">
                                                        <span>1B</span>
                                                    </a>).
                                                </p><p id="__p9" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">以5pmol 的双链 DNA 模板为模板，通过 PCR 扩增，制备了用于筛选的幼稚文库。由于每个模板包含19个随机位置，该库的理论复杂度为3 × 1011个唯一序列。一个5 pmol 的样本包含每个唯一序列的10个拷贝。PCR 反应(100μl)包括20mm Tris ph8.4,50mm KCl，4mm MgCl2,200nm dNTPs，每个引物400nm，20nm DNA 模板和0.1 u/μl 重组 Taq 聚合酶(Invitrogen)。以8% 聚丙烯酰胺凝胶(29:1丙烯酰胺: 双丙烯酰胺)为载体，在0.5 × 双硼酸乙二胺(TBE)缓冲液中电泳，用溴化乙锭染色，定性检测扩增产物，确定最佳 PCR 循环次数。在95 ° c 温度下进行2min 的 PCR 变性，在95 ° c 温度下进行30s 变性，在52 ° c 温度下进行30s 退火，在72 ° c 温度下进行30s 延伸，在95 ° c 温度下进行30s 变性，在65 ° c 温度下进行30s 退火，在72 ° c 温度下进行30s 延伸，在72 ° c 温度下进行5min 延伸。产物在1 × TAE 缓冲液中经2.5% 琼脂糖凝胶电泳分离，主要产物用快速凝胶提取试剂盒(洽根)纯化。初始 RNA 池是从6 pmol 的 DNA 模板(代表∼1012序列)转录使用幅标 T7转录试剂盒(震中)如上所述。RNA 纯化采用10% 聚丙烯酰胺凝胶电泳(29:1丙烯酰胺: 双丙烯酰胺，8 m 尿素，0.5 x TBE) ，紫外阴影显影，在2 × PK 缓冲液(200mm Tris ph7.6,2.5 mM 乙二胺四乙酸(EDTA) ，300mm NaCl 和2% 十二烷基硫酸钠)中室温搅拌下连夜提取和洗脱。用等体积的苯酚/氯仿(1:1，v/v)提取洗脱物，加入2.5体积的乙醇沉淀 RNA。对于初始选择，在适当的选择缓冲液(SB)中，用23 pmol 的池 RNA 代表3 × 1011 DNA 模板的∼43个拷贝，在体积为50ml 的溶液中加入2nm 的目标蛋白，使其不具有催化活性: BamHI (E111A) : 10mm HEPES pH 7.5,400mm NaCl，1mm MgCl2; KpnI: 10mm HEPES pH 7.5,100mm NaCl，1mm l 2; PacI: 10hpes pH 7.5,10mm NaCl，1mm l 2。在筛选之前，所有 RNA 池都在70 ° c 的 SB 中孵育5分钟，然后冷却至室温以促进 RNA 折叠。在有或无蛋白的 SB 缓冲液中将 rna 孵育45min，然后用硝酸纤维素过滤器(Millipore HAWP02500,0.45 μm HA 过滤器)进行真空分配，进行筛选。未结合的 rna 用两个10ml SB 的分量从过滤器中冲洗出来。然后，捕获的 RNA 被切伦科夫辐射定量，该辐射通过使用闪烁计数器的3 h 设置过滤器检测到。结合 RNA 在500μl 2 × PK 缓冲液中65 ° c 洗脱30min。以40μg 糖原为载体，用等体积的酚: 氯仿(1:1)和乙醇沉淀的 RNA 提取洗脱液。在第3轮和随后的单轮负选择之前，将 RNA 文库置于硝酸纤维素滤纸条中，处理上清液进行选择。利用5′-A5TATG2A2CGCT2CATGCAT 引物 LJM-4486反转录获得三分之一的 RNA。在添加 MMLV 逆转录酶和 dNTPs 之前，引物在室温下分别在65 ° c 和5min 温箱中退火5min。每个40μl 反转录反应包括1.25 μm 引物 LJM-4486、400μM dNTPs、50mm KCl、6mm MgCl2、50mm Tris ph8.0和2μl 200u/μl MMLV 逆转录酶(Invitrogen.)反应在室温下孵育10分钟，在37 °c 孵育30分钟。每个反应的一半用于优化 PCR。PCR 扩增剩余 cDNA 后，用 PCR 纯化试剂盒(Qiagen)纯化 DNA 文库。为了准备后续轮选择的 RNA 池，200纳克(4 pmol)纯化 DNA 转录如前所述。为了监测放射标记 RNA 在每一轮选择中的结合和恢复情况，将硝化纤维素过滤放射性与总放射性的一小部分进行比较，以确定与蛋白质靶标结合的 RNA 部分。每种酶进行二十轮选择，BamHI 在第14周期达到最大 RNA 回收率，KpnI 在第15周期，PacI 在第19周期(图1B)。</p>
                                            </div>
                                            <div id="SEC2-4" class="sec">
                                                <h3 id="SEC2-4title" class="cyxy-trs-source cyxy-trs-source-ted">Cloning</h3><h3 id="SEC2-4title" class="cyxy-trs-source cyxy-trs-target">克隆</h3>
                                                <p id="__p10" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted">The double-stranded DNA libraries of rounds 14, 16 and 18 for BamHI (E111A), 15, 16 and 18 for KpnI, and 18, 19 and 20 for PacI were ligated into the pGEM-TEasy vector using a TA cloning kit (Promega). Clones with coherent sequences were aligned using multiple sequence alignment tool MUSCLE (EMBL).</p><p id="__p10" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">用 TA 克隆试剂盒(Promega)将 BamHI (E111A)14、16和18、 KpnI 15、16和18、 PacI 18、19和20的双链 DNA 文库连接到 pGEM-TEasy 载体中。使用多重序列比对工具 MUSCLE (EMBL)对具有相干序列的克隆进行比对。</p>
                                            </div>
                                            <div id="SEC2-5" class="sec">
                                                <h3 id="SEC2-5title" class="cyxy-trs-source cyxy-trs-source-ted">Qualitative binding screen</h3><h3 id="SEC2-5title" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">定性捆绑屏幕</h3>
                                                <p id="__p11" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted">RNAs were 3′ end-labeled by ligation of [<sup>32</sup>P]-cytidine 3′,5′ bis(phosphate), (3000 Ci/mmol; 10 mCi/ml), using T4 RNA ligase (New England Biolabs). Unincorporated nucleotides where removed by size exclusion chromatography using Chromaspin TE-10 gel filtration columns (Clontech). Binding reactions (25 μl) contained the appropriate SB, 4 nM RNA and 20 nM target protein. Reactions were incubated for 30 min at 37°C. Binding reactions were supplemented with glycerol loading dye and analyzed by electrophoresis through 8% polyacrylamide gels in 1× TBE at 3W for 1 h. Gels where then dried and exposed overnight using storage phosphor imaging technology (Typhoon FLA-7000).</p><p id="__p11" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">用 T4 RNA 连接酶连接[32P ]-胞苷3′ ，5′-二(磷酸) ，(3000ci/mmol; 10mci/ml) ，标记3′端。用 Chromaspin TE-10凝胶过滤柱(cloontech)通过尺寸排阻色谱法去除非合并的核苷酸。结合反应(25μl)包含适当的 SB、4nm RNA 和20nm 靶蛋白。反应在37 °c 孵育30分钟。结合反应用甘油负载染料进行补充，并用8% 聚丙烯酰胺凝胶电泳分析。</p>
                                            </div>
                                            <div id="SEC2-6" class="sec">
                                                <h3 id="SEC2-6title" class="cyxy-trs-source cyxy-trs-source-ted">Quantitative binding assay</h3><h3 id="SEC2-6title" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">定量结合分析</h3>
                                                <p id="__p12" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted">RNA aptamer binding to target REases was determined by electrophoretic gel mobility shift assay. RNAs were transcribed from the appropriate double-stranded DNA template and radiolabeled as described above. Binding reactions contain radiolabeled aptamer (13 nM) and the indicated concentration of KpnI in New England Biolabs 1.1 buffer (10 mM Bis-Tris-Propane-HCl, pH 7, 10 mM MgCl<sub>2</sub>, 100 μg/ml bovine serum albumin (BSA)) for 30 min at 37°C. Samples where then directly loaded onto 8% polyacrylamide gels (19:1 acrylamide:bisacrylamide) and electrophoresed in 1× TBE buffer. Gels were dried for 30 min and signal detected by storage phosphor imaging. Image quantitation involved ImageJ software (NIH) with the equilibrium dissociation constant estimated by least squares fitting to a modified Hill equation using R software (version 2.13.0):</p><p id="__p12" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">凝胶迁移率变化法测定 RNA 适配子与靶 RNA 的结合。Rna 被转录自适当的双链 DNA 模板和放射性标记如上所述。结合反应包括放射性核酸适配体(13nm)和 KpnI 在新英格兰 Biolabs 1.1缓冲液(10mm Bis-Tris-Propane-HCl，pH 7,10mm MgCl2,100μg/ml 牛血清白蛋白)中的指示浓度，在37 ° c 下持续30min。样品直接负载于8% 聚丙烯酰胺凝胶(19:1丙烯酰胺: 双丙烯酰胺)上，在1 × TBE 缓冲液中电泳。凝胶干燥30min，储存磷光体成像检测信号。图像定量使用了 ImageJ 软件(NIH) ，用 r 软件用最小二乘法拟合修正的 Hill 方程(版本2.13.0)来估计平衡解离常数:</p>
                                                <div class="disp-formula" id="M1">
                                                    <div class="f">
                                                        <img src="" class="equation va_middle inline_block" alt="equation M8" title="equation M8">
                                                    </div>
                                                    <div class="l cyxy-trs-source">(1)</div>
                                                </div>
                                                <p class="cyxy-trs-source cyxy-trs-source-ted">
                                                    where <em>θ</em> is the fraction of RNA bound, <em>K</em>
                                                    <sub>d</sub> is the dissociation constant, <em>P</em>
                                                    <sub>t</sub> is the total protein concentration and <em>h</em> is the Hill coefficient.
                                                </p><p class="cyxy-trs-source cyxy-trs-target" contenteditable="true">其中 θ 是 RNA 结合的部分，k d 是解离常数，p t 是总蛋白质浓度，h 是希尔系数。</p>
                                            </div>
                                            <div id="SEC2-7" class="sec">
                                                <h3 id="SEC2-7title" class="cyxy-trs-source cyxy-trs-source-ted">Restriction inhibition assay</h3><h3 id="SEC2-7title" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">限制性抑制试验</h3>
                                                <p id="__p13" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted">A fluorescent 321-bp double-stranded DNA probe was prepared by PCR from five overlapping oligonucleotides and two primers synthesized with 6-FAM fluorescein phosphoramidites. The probe contains unique recognition sites for BamHI, KpnI and PacI in an asymmetric pattern (Figure 2C). Qualitative inhibition assays included RNA aptamer (40 μM) in the digestion buffer recommended by the enzyme manufacturer (New England Biolabs): buffer 3.1 for BamHI, buffer 1.1 for KpnI and CutSmart buffer for PacI. Reaction buffer with RNA aptamers were heated to 70°C for 5 min then allowed to cool to room temperature. The appropriate commercial REase was then added to a final concentration of 0.2 U/μl as indicated and reactions incubated for 30 min at 37°C. Fluorescent 321-bp target probe (20 nM) was added and digestion allowed to proceed for 30 min at 37°C. Reactions were terminated by heating for 15 min at 65°C. After addition of glycerol loading dye, samples were analyzed by electrophoresis through 8% polyacrylamide gels (29:1 acylamide:bisacylamide) in 0.5× TBE buffer. Reaction products were imaged using the Typhoon FLA-7000 instrument at a wavelength of 475 nm. When desired, assay quantitation was implemented by image analysis (Image J) and the RNA aptamer concentration necessary for 50% enzyme inhibition (IC<sub>50</sub>) estimated by curve fitting:</p><p id="__p13" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">以5个重叠的寡核苷酸和2个以6-FAM 荧光素磷酰胺为底物的引物，采用 PCR 方法制备了321bp 双链 DNA 探针。该探针包含不对称模式下 BamHI、 KpnI 和 PacI 的唯一识别位点(图2C)。定性抑制试验包括酶制剂生产商推荐的消化缓冲液中的 RNA 适配子(40μM) : BamHI 缓冲液3.1，KpnI 缓冲液1.1，PacI 缓冲液 CutSmart 缓冲液。反应缓冲液与 RNA 适配体加热至70 ° c 5min，然后冷却至室温。最终浓度为0.2 u/μl，在37 ° c 下反应30 min。加入荧光321bp 靶探针(20nm) ，在37 °c 下进行30min 消解。反应在65 ° c 加热15分钟后终止。样品加入甘油负载染料后，在0.5 × TBE 缓冲液中用8% 聚丙烯酰胺凝胶电泳进行分析。利用台风 FLA-7000仪在475 nm 波长处对反应产物进行了成像。如果需要，通过图像分析(图 j)和用曲线拟合法估算的50% 酶抑制(IC50)所需的 RNA 适配子浓度来实现分析定量:</p>
                                                <div class="disp-formula" id="M2">
                                                    <div class="f">
                                                        <img src="" class="equation va_middle inline_block" alt="equation M9" title="equation M9">
                                                    </div>
                                                    <div class="l cyxy-trs-source">(2)</div>
                                                </div>
                                                <p class="cyxy-trs-source cyxy-trs-source-ted">where <em>f</em> is the fraction of active enzyme, <em>S</em> is the slope factor and <em>A</em> is the aptamer concentration.</p><p class="cyxy-trs-source cyxy-trs-target" contenteditable="true">其中 f 为活性酶的分数，s 为斜率因子，a 为适配子浓度。</p>
                                            </div>
                                            <div id="SEC2-8" class="sec">
                                                <h3 id="SEC2-8title" class="cyxy-trs-source cyxy-trs-source-ted"><mark id="mark-id-4" class="tagged tagged-node" c-type="T" c-id="4" c-name="AptamerName" style="background-color: rgb(204, 255, 255);">{</mark>Curve-fitting<mark class="tagged tagged-node" c-type="T" c-id="4" c-name="AptamerName" style="background-color: rgb(204, 255, 255);">}</mark></h3><h3 id="SEC2-8title" class="cyxy-trs-source cyxy-trs-target">曲线拟合</h3>
                                                <p id="__p14" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted">
                                                    A weighted non-linear least squares method was used to minimize the cost function <em>C</em>(<strong>p</strong>) with respect to the free parameters <strong>p</strong>
                                                </p><p id="__p14" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">采用加权非线性最小二乘法，使费用函数 c (p)相对于自由参数 p 极小化</p>
                                                <div class="disp-formula" id="M3">
                                                    <div class="f">
                                                        <img src="" class="equation va_middle inline_block" alt="equation M10" title="equation M10">
                                                    </div>
                                                    <div class="l cyxy-trs-source">(3)</div>
                                                </div>
                                                <p class="cyxy-trs-source cyxy-trs-source-ted">
                                                    where <em>M</em> is the number of data points being fit, and (corresponding to the <em>m</em>-th data point) <em>
                                                        c<sub>m</sub>
                                                    </em> are concentrations, <img src="" class="equation va_middle inline_block" alt="equation M11" title="equation M11"> are experimentally determined mean values, <img src="" class="equation va_middle inline_block" alt="equation M12" title="equation M12"> are standard deviations of <img src="" class="equation va_middle inline_block" alt="equation M13" title="equation M13"> and <img src="" class="equation va_middle inline_block" alt="equation M14" title="equation M14"> are theoretical predictions of the model (either Hill equation or 2-parameter logistic model).
                                                </p><p class="cyxy-trs-source cyxy-trs-target" contenteditable="true">其中 m 是拟合的数据点的数目，和(对应的第 m 个数据点) cm 是浓度，是实验确定的平均值，是标准偏差，是模型的理论预测(希尔方程或双参数逻辑斯谛模型)。</p>
                                            </div>
                                            <div id="SEC2-9" class="sec">
                                                <h3 id="SEC2-9title" class="cyxy-trs-source cyxy-trs-source-ted">Monte Carlo estimation of uncertainty</h3><h3 id="SEC2-9title" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">不确定度的蒙特卡罗估计</h3>
                                                <p id="__p15" class="p p-first cyxy-trs-source cyxy-trs-source-ted">
                                                    Simulated datasets were generated from the experimental data by adding Gaussian noise, i.e. randomly sampling from the normal distributions <img src="" class="equation va_middle inline_block" alt="equation M15" title="equation M15">, corresponding to the experimental mean and standard deviations of the <em>m</em>-th data point. These simulated data sets (<img src="" class="equation va_middle inline_block" alt="equation M16" title="equation M16"> for <img src="" class="equation va_middle inline_block" alt="equation M17" title="equation M17">) were then minimized against the Hill equation (for <em>K</em>
                                                    <sub>d</sub>) or 2-parameter logistic model (for IC<sub>50</sub>) to determine <img src="" class="equation va_middle inline_block" alt="equation M18" title="equation M18">
                                                </p><p id="__p15" class="p p-first cyxy-trs-source cyxy-trs-target" contenteditable="true">通过加入高斯噪声，即从正态分布中随机抽样，对应于 m 次数据点的实验均值和标准偏差，从实验数据中生成模拟数据集。这些模拟数据集(为)然后最小化对希尔方程(为 k d)或双参数逻辑斯谛模型(为 IC50)来确定</p>
                                                <div class="disp-formula" id="M4">
                                                    <div class="f">
                                                        <img src="" class="equation va_middle inline_block" alt="equation M19" title="equation M19">
                                                    </div>
                                                    <div class="l cyxy-trs-source">(4)</div>
                                                </div>
                                                <p class="cyxy-trs-source"></p>
                                                <p id="__p16" class="cyxy-trs-source cyxy-trs-source-ted">
                                                    For a given parameter <em>
                                                        p<sub>k</sub>
                                                    </em>, the standard deviation was estimated as:
                                                </p><p id="__p16" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">对于给定的参数 pk，标准差估计为:</p>
                                                <div class="disp-formula" id="M5">
                                                    <div class="f">
                                                        <img src="" class="equation va_middle inline_block" alt="equation M20" title="equation M20">
                                                    </div>
                                                    <div class="l cyxy-trs-source">(5)</div>
                                                </div>
                                                <p class="cyxy-trs-source cyxy-trs-source-ted">
                                                    where <em>ε</em>
                                                    <sub>max</sub> was set at 10 000 and <img src="" class="equation va_middle inline_block" alt="equation M21" title="equation M21"> (the mean value) is:
                                                </p><p class="cyxy-trs-source cyxy-trs-target" contenteditable="true">其中 ε 最大设置为10000，(平均值)是:</p>
                                                <div class="disp-formula" id="M6">
                                                    <div class="f">
                                                        <img src="" class="equation va_middle inline_block" alt="equation M22" title="equation M22">
                                                    </div>
                                                    <div class="l cyxy-trs-source">(6)</div>
                                                </div>
                                                <p class="cyxy-trs-source"></p>
                                                <p id="__p17" class="cyxy-trs-source cyxy-trs-source-ted">
                                                    Additionally, the relative bias in the parameter <em>
                                                        p<sub>k</sub>
                                                    </em> is given by,
                                                </p><p id="__p17" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">另外，参数 pk 的相对偏差由,</p>
                                                <div class="disp-formula" id="M7">
                                                    <div class="f">
                                                        <img src="" class="equation va_middle inline_block" alt="equation M23" title="equation M23">
                                                    </div>
                                                    <div class="l cyxy-trs-source">(7)</div>
                                                </div>
                                                <p class="cyxy-trs-source"></p>
                                                <p id="__p18" class="p p-last cyxy-trs-source cyxy-trs-source-ted">
                                                    For all of the parameters, the median relative bias was 2.4%, giving confidence in the method. For <em>K</em>
                                                    <sub>d</sub> and IC<sub>50</sub>, the median relative biases were 0.5 and 3.8%, respectively.
                                                </p><p id="__p18" class="p p-last cyxy-trs-source cyxy-trs-target" contenteditable="true">对于所有的参数，中位数相对偏差为2.4% ，给予方法的信心。对于 kd 和 IC50，中位数相对偏差分别为0.5% 和3.8% 。</p>
                                            </div>
                                            <div id="SEC2-10" class="sec">
                                                <h3 id="SEC2-10title" class="cyxy-trs-source cyxy-trs-source-ted">In-line probing</h3><h3 id="SEC2-10title" class="cyxy-trs-source cyxy-trs-target">内联探测</h3>
                                                <p id="__p19" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted">RNA aptamers were dephosphorylated and labeled at 5′ termini using γ- [<sup>32</sup>P]-ATP (6000 Ci/mmole; 10 mCi/ml), using calf intestinal alkaline phosphatase (CIP) and T4 polynucleotide kinase (New England Biolabs), followed by gel purification. All reactions contained 10<sup>5</sup> radioactive counts per minute of radiolabeled aptamer RNA. In-line probing reactions (10 μl) contained 50 mM Tris–HCl (pH 8.3 at 20°C), 20 mM MgCl<sub>2</sub> and 100 mM KCl. Reactions were incubated at room temperature for 24 or 48 h. RNase T1 treatment was performed in 25 mM sodium citrate pH 5.0, 0.1 U/μl RNase T1, in the presence of 5 M urea and 0.75 mM EDTA (pH 8.0 at 20°C). Reactions were incubated at 55°C for 3 min. Partial alkaline hydrolysis reactions were performed in 50 mM Na<sub>2</sub>CO<sub>3</sub> pH 9.0, 1 mM EDTA, incubated at 95°C for 5 min. All reactions were terminated by ethanol precipitation. Vacuum-dried pellets were then resuspended in 8 μl deionized formamide gel-loading solution and immediately analyzed by gel electrophoresis through 10% denaturing polyacrylamide sequencing gels (10%, 29:1 acrylamide:bisacrylamide) in 1× TBE buffer. Electrophoresis was at 48 W for 3 h. Gels were then transferred to blotting paper, dried for 1 h and signal detected by storage phosphor imaging. Signal analysis was performed using SAFA (<a rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922015">26</a>). Extracted and quantitated band intensities were then normalized by subtracting background from unmodified sample and dividing by the partial hydrolysis signal, yielding values to generate a colored heat plot.</p><p id="__p19" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">用 γ-[32P ]-ATP (6000ci/mmole; 10mci/ml)对 RNA 适配子进行去磷酸化和5′端标记，然后用小牛肠道碱性磷酸酶(CIP)和 t 4多核苷酸激酶(New England biolab)进行凝胶纯化。所有反应都含有105个放射性核酸适配每分钟计数 RNA。在线探测反应(10μl)包括50mm Tris-HCl (ph8.3,20 ° c) ，20mm MgCl2和100mm KCl。反应在室温下进行，温度分别为24和48h，反应时间分别为25mm 柠檬酸钠 ph5.0,0.1 u/μl RNase T1,5m 尿素和0.75 mM EDTA (ph8.0，温度为20 °c)。反应在55 ° c 温度下孵育3min。在50mm Na2CO3 ph9.0,1mm EDTA 中进行部分水焚葬反应，在95 ° c 培养5min。所有反应均用乙醇沉淀终止。将真空干燥的小丸悬浮在8 μl 去离子甲酰胺凝胶负载溶液中，然后在1 × TBE 缓冲液中，通过10% 变性聚丙烯酰胺测序凝胶(10% ，29:1丙烯酰胺: 双丙烯酰胺)立即进行凝胶电泳分析。凝胶电泳在48w 下进行3h，然后转移到吸墨纸上，干燥1h，用储存磷光体成像检测信号。使用 SAFA (26)进行信号分析。提取和定量的波段强度，然后归一化的背景从未修改的样品和除以部分水解信号，屈服值产生一个彩色热图。</p>
                                            </div>
                                            <div id="SEC2-11" class="sec">
                                                <h3 id="SEC2-11title" class="cyxy-trs-source cyxy-trs-source-ted">Competition gel shift assay</h3><h3 id="SEC2-11title" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">竞争凝胶移位法</h3>
                                                <p id="__p20" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted">Binding reactions were performed in digestion inhibition conditions (10 mM HEPES pH 7.0, 100 mM NaCl, 1 mM Ca<sup>2+</sup> and 1 μg/ml BSA). Reaction buffer with RNA aptamers was heated to 70°C for 5 min and allowed to cool to room temperature. The appropriate commercial REase was then added to a final concentration of 4 U/μl and reactions incubated for 30 min at 37°C. Fluorescent DNA probe (20 nM) was then added with incubation for 30 min at 37°C. After addition of glycerol loading dye, samples were immediately analyzed by electrophoresis through 8% polyacrylamide gels (29:1 acylamide:bisacylamide) in 0.5× TBE buffer. Gels were imaged using the Typhoon FLA-7000 instrument at a wavelength of 475 nm.</p><p id="__p20" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">结合反应在消化抑制条件下进行(10mm HEPES ph7.0,100mm NaCl，1mm Ca2 + 和1μg/ml BSA)。与 RNA 适配体反应缓冲液加热至70 ° c，冷却至室温。然后在最终浓度为4u/μl 时加入适当的商品化 REase，在37 °c 下反应30min。荧光 DNA 探针(20nm)加入37 °c 孵育30min。样品加入甘油负载染料后，立即在0.5 × TBE 缓冲液中用8% 聚丙烯酰胺凝胶电泳进行分析。凝胶的成像使用台风 FLA-7000仪器在475纳米的波长。</p>
                                            </div>
                                            <div id="SEC2-12" class="sec sec-last">
                                                <h3 id="SEC2-12title" class="cyxy-trs-source cyxy-trs-source-ted">CD spectroscopy</h3><h3 id="SEC2-12title" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">CD 光谱学</h3>
                                                <p id="__p21" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted">
                                                    RNA samples were transcribed <em>in</em>
                                                    <em>vitro</em> from synthetic duplex DNA templates as previously described, gel purified and extracted from gels using an Elutrap electroelution system, buffer exchanged into 100 mM sodium cacodylate buffer, pH 6.6 and concentrated to ∼300 ng/μl. Circular dichroism spectra were collected using a JASCO J-810 spectropolarimeter. Wavelength scans from 200 to 320 nm were collected every 1 nm at a speed of 100 nm/min at a sensitivity of 100 mdeg with accumulation of 5 counts at 25°C using a 1 mm cuvette.
                                                </p><p id="__p21" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">采用体外合成的双链 DNA 模板转录 RNA 样品，用洗脱电流洗脱法提取凝胶，用100mm 的 cacodylate 缓冲液交换缓冲液，pH 6.6，浓缩至∼300ng/μl。用 JASCO J-810分光偏振仪收集了圆二色性光谱。以100nm/min 的速度，在100mdeg 的灵敏度下，每1nm 采集200ー320nm 的波长扫描，并在25 ° c 下用1mm 的试管累积5个波长。</p>
                                            </div>
                                        </div>
                                        <div id="SEC3" class="tsec sec">
                                            <h2 class="head no_bottom_margin cyxy-trs-source cyxy-trs-source-ted" id="SEC3title" style="text-transform: uppercase;">RESULTS</h2><h2 class="head no_bottom_margin cyxy-trs-source cyxy-trs-target" id="SEC3title" style="text-transform: uppercase;">结果</h2>
                                            <div id="SEC3-1" class="sec sec-first">
                                                <h3 id="SEC3-1title" class="cyxy-trs-source cyxy-trs-source-ted">
                                                    <em>In</em>
                                                    <em>vitro</em> selection
                                                </h3><h3 id="SEC3-1title" class="cyxy-trs-source cyxy-trs-target">体外选择</h3>
                                                <p id="__p22" class="p p-first cyxy-trs-source cyxy-trs-source-ted">
                                                    SELEX against Type II REases BamHI, KpnI and PacI was performed using a 67-nt RNA library containing blocks of 10 and 9 random nucleotides flanking a 5′-GUAA tetraloop (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A)</span>
                                                        </span>
                                                        <span>1A</span>
                                                    </a>) within a weak stem-loop structure modeled after the framework of the previously-studied anti-NF-κB p50 RNA aptamer [(<a rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921993">12</a>,<a rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921996">17</a>,<a rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921998">23</a>) and see ‘Materials and Methods’ section]. To suppress potential RNA aptamer cleavage by KpnI and PacI, selections were performed in the presence of Ca<sup>2+</sup>. Selections with the catalytically inactive BamHI (E111A) mutant were performed in Mg<sup>2+</sup>. Preliminary experiments revealed that each enzyme required different salt conditions to maintain selection stringency (fraction of recovered RNAs less than 1% in early rounds; data not shown). These individual conditions were then used for each enzyme throughout the 20 rounds of selection. A negative selection against nitrocellulose filter strips was performed after odd cycles to reduce aptamers enriched for binding to the selection matrix. After 20 rounds we studied aptamers from three different rounds of selection including the most enriched cycles for each REase. Anti-BamHI, anti-KpnI and anti-PacI RNA aptamers were retained at 36, 26 and 9% after 14, 15 and 19 selection cycles, respectively (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1B).</span>
                                                        </span>
                                                        <span>1B</span>
                                                    </a>). After cloning, complete sequences were obtained for ∼50 aptamers from each of the sequenced cycles and analyzed using sequence alignment tool MUSCLE (EMBL) (Table <a target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid="">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">(Table1).</span>
                                                        </span>
                                                        <span>1</span>
                                                    </a>). The number of unique sequences in each library tended to decrease with more cycles, as expected. Interestingly, although selections against BamHI gave the highest fraction of RNA recovery (36% in cycle 14), 41 unique sequences were found in that pool and the aptamer with highest abundance was found only three times. In contrast, cycle 15 of KpnI selection gave only 9 unique sequences and the most abundant was found 19 times. Thus, libraries were not equally converged (Table <a target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid="">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">(Table1).</span>
                                                        </span>
                                                        <span>1</span>
                                                    </a>). The basis for this difference is not known, but different protein characteristics and ionic conditions of selections presumably played roles in the effect.
                                                </p><p id="__p22" class="p p-first cyxy-trs-source cyxy-trs-target" contenteditable="true">本研究利用含有10和9个随机核苷酸片段的67 nt RNA 文库(图(图1A)) ，模拟先前研究的反 nf-κb p50 RNA 适配子((12,17,23))的框架，在弱茎环结构中进行 SELEX 抗 II 型糖尿病(II 型糖尿病)的研究。为了抑制 KpnI 和 PacI 对 RNA 适配子的切割，我们在 Ca2 + 存在的情况下进行了选择。在 Mg2 + 培养基上进行了催化活性缺失的 BamHI (E111A)突变体的筛选。初步实验表明，每种酶需要不同的盐条件来保持选择的严格性(在早期轮中恢复的 rna 小于1% ; 数据没有显示)。这些单独的条件然后用于每个酶在整个20轮选择。单次循环后对硝化纤维素过滤条进行负选择，以减少适配子与选择矩阵的结合。经过20轮的筛选，我们研究了三轮不同的选择中的适配体，包括每个 REase 的最丰富的选择周期。经过14、15和19个选择周期后，抗 bamhi、抗 kpni 和抗 paci RNA 适配子分别保留在36、26和9% (图1b)。1B).克隆后，从每个测序的循环中获得∼50个适配体的完整序列，并用序列比对工具 MUSCLE (EMBL)进行分析(表1)。1).正如预期的那样，每个库中唯一序列的数量随着循环次数的增加而减少。有趣的是，尽管针对 BamHI 的选择提供了最高的 RNA 提取率(第14周期为36%) ，但在该库中发现了41个独特的序列，最高丰度的适配体只发现了三次。相比之下，KpnI 选择的第15周期只有9个唯一序列，最丰富的序列有19个。因此，库并没有同样地聚合(表(表1))。1).这种差异的基础尚不清楚，但不同的蛋白质特性和离子条件的选择可能在其中起了作用。</p>
                                                <!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1">
                                                    <h3 class="cyxy-trs-source cyxy-trs-source-ted">Table 1.</h3><h3 class="cyxy-trs-source cyxy-trs-target">表一</h3>
                                                    <!--caption a7--><div class="caption">
                                                        <!--caption a8--><strong class="cyxy-trs-source">Dominant aptamer sequences for each of three target REases<font class="cyxy-trs-target" contenteditable="true"> 三个目标染色体的优势核酸适配子序列</font></strong>
                                                    </div>
                                                    <div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607996152928" class="xtable">
                                                        <table frame="hsides" rules="groups" class="rendered small default_table">
                                                            <thead>
                                                                <tr>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">Target<font class="cyxy-trs-target"> 目标</font></th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">Round<font class="cyxy-trs-target"> 圆形</font></th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">Sequenced clones<font class="cyxy-trs-target"> 测序克隆</font></th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">Unique sequences<font class="cyxy-trs-target" contenteditable="true"> 独特的序列</font></th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">Abundance of most selected sequence<font class="cyxy-trs-target" contenteditable="true"> 大多数选择序列的丰度</font></th>
                                                                    <th align="left" rowspan="1" colspan="1" class="cyxy-trs-source">Random regions of dominant aptamers<font class="cyxy-trs-target" contenteditable="true"> 优势核酸适体的随机区域</font></th>
                                                                </tr>
                                                            </thead>
                                                            <tbody>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="left" rowspan="1" colspan="1">
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">NNNNNNNNNN<font class="cyxy-trs-target"> 神经网络</font></kbd>
                                                                        </strong>
                                                                        <kbd class="cyxy-trs-source">GUAA</kbd>
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">NNNNNNNNN<font class="cyxy-trs-target"> 中文名称</font></kbd>
                                                                        </strong>
                                                                    </td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">
                                                                        <strong class="cyxy-trs-source">BamHI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></strong>
                                                                    </td>
                                                                    <td align="center" rowspan="1" colspan="1">
                                                                        14<sup>a</sup>
                                                                    </td>
                                                                    <td align="center" rowspan="1" colspan="1">48</td>
                                                                    <td align="center" rowspan="1" colspan="1">41</td>
                                                                    <td align="center" rowspan="1" colspan="1">3</td>
                                                                    <td align="left" rowspan="1" colspan="1">
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">AAUAAAUCGA</kbd>
                                                                        </strong>
                                                                        <kbd class="cyxy-trs-source">GUAA</kbd>
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">UAAUCUAAU</kbd>
                                                                        </strong>
                                                                    </td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="center" rowspan="1" colspan="1">16</td>
                                                                    <td align="center" rowspan="1" colspan="1">50</td>
                                                                    <td align="center" rowspan="1" colspan="1">37</td>
                                                                    <td align="center" rowspan="1" colspan="1">5</td>
                                                                    <td align="left" rowspan="1" colspan="1">
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">UAUAUAUCAA</kbd>
                                                                        </strong>
                                                                        <kbd class="cyxy-trs-source">GUAA</kbd>
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">UAAUCAUAA</kbd>
                                                                        </strong>
                                                                    </td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="center" rowspan="1" colspan="1">18</td>
                                                                    <td align="center" rowspan="1" colspan="1">48</td>
                                                                    <td align="center" rowspan="1" colspan="1">36</td>
                                                                    <td align="center" rowspan="1" colspan="1">5</td>
                                                                    <td align="left" rowspan="1" colspan="1">
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">AAAUCGUAUA<font class="cyxy-trs-target" contenteditable="true"> Aaucguaua</font></kbd>
                                                                        </strong>
                                                                        <kbd class="cyxy-trs-source">GUAA</kbd>
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">UAAUUAGUA</kbd>
                                                                        </strong>
                                                                    </td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">
                                                                        <strong class="cyxy-trs-source">KpnI</strong>
                                                                    </td>
                                                                    <td align="center" rowspan="1" colspan="1">
                                                                        15<sup>a</sup>
                                                                    </td>
                                                                    <td align="center" rowspan="1" colspan="1">54</td>
                                                                    <td align="center" rowspan="1" colspan="1">9</td>
                                                                    <td align="center" rowspan="1" colspan="1">19</td>
                                                                    <td align="left" rowspan="1" colspan="1">
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">GGCGAAAAGC<font class="cyxy-trs-target" contenteditable="true"> 自动增益控制器</font></kbd>
                                                                        </strong>
                                                                        <kbd class="cyxy-trs-source">GUAA</kbd>
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">CCGCGGCGA<font class="cyxy-trs-target" contenteditable="true"> 中国地球物理学会</font></kbd>
                                                                        </strong>
                                                                    </td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="center" rowspan="1" colspan="1">16</td>
                                                                    <td align="center" rowspan="1" colspan="1">52</td>
                                                                    <td align="center" rowspan="1" colspan="1">8</td>
                                                                    <td align="center" rowspan="1" colspan="1">15</td>
                                                                    <td align="left" rowspan="1" colspan="1">
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">GGCGAAAAGC<font class="cyxy-trs-target" contenteditable="true"> 自动增益控制器</font></kbd>
                                                                        </strong>
                                                                        <kbd class="cyxy-trs-source">GUAA</kbd>
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">CCGCGGCGA<font class="cyxy-trs-target" contenteditable="true"> 中国地球物理学会</font></kbd>
                                                                        </strong>
                                                                    </td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="center" rowspan="1" colspan="1">18</td>
                                                                    <td align="center" rowspan="1" colspan="1">58</td>
                                                                    <td align="center" rowspan="1" colspan="1">8</td>
                                                                    <td align="center" rowspan="1" colspan="1">15</td>
                                                                    <td align="left" rowspan="1" colspan="1">
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">GGCGAAAAGC<font class="cyxy-trs-target" contenteditable="true"> 自动增益控制器</font></kbd>
                                                                        </strong>
                                                                        <kbd class="cyxy-trs-source">GUAA</kbd>
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">CCGCGGCGA<font class="cyxy-trs-target" contenteditable="true"> 中国地球物理学会</font></kbd>
                                                                        </strong>
                                                                    </td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">
                                                                        <strong class="cyxy-trs-source">PacI</strong>
                                                                    </td>
                                                                    <td align="center" rowspan="1" colspan="1">18</td>
                                                                    <td align="center" rowspan="1" colspan="1">46</td>
                                                                    <td align="center" rowspan="1" colspan="1">12</td>
                                                                    <td align="center" rowspan="1" colspan="1">18</td>
                                                                    <td align="left" rowspan="1" colspan="1">
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">AUAUAUAAUU<font class="cyxy-trs-target"> 奥奥奥乌</font></kbd>
                                                                        </strong>
                                                                        <kbd class="cyxy-trs-source">GUAA</kbd>
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">AUUCUAAUG<font class="cyxy-trs-target" contenteditable="true"> Auucuaug</font></kbd>
                                                                        </strong>
                                                                    </td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="center" rowspan="1" colspan="1">
                                                                        19<sup>a</sup>
                                                                    </td>
                                                                    <td align="center" rowspan="1" colspan="1">36</td>
                                                                    <td align="center" rowspan="1" colspan="1">13</td>
                                                                    <td align="center" rowspan="1" colspan="1">12</td>
                                                                    <td align="left" rowspan="1" colspan="1">
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">AUAUAUAAUU<font class="cyxy-trs-target"> 奥奥奥乌</font></kbd>
                                                                        </strong>
                                                                        <kbd class="cyxy-trs-source">GUAA</kbd>
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">AUUCUAAUG<font class="cyxy-trs-target" contenteditable="true"> Auucuaug</font></kbd>
                                                                        </strong>
                                                                    </td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="center" rowspan="1" colspan="1">20</td>
                                                                    <td align="center" rowspan="1" colspan="1">54</td>
                                                                    <td align="center" rowspan="1" colspan="1">16</td>
                                                                    <td align="center" rowspan="1" colspan="1">13</td>
                                                                    <td align="left" rowspan="1" colspan="1">
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">AUAUAUAAUU<font class="cyxy-trs-target"> 奥奥奥乌</font></kbd>
                                                                        </strong>
                                                                        <kbd class="cyxy-trs-source">GUAA</kbd>
                                                                        <strong>
                                                                            <kbd class="cyxy-trs-source">AUUCUAAUG<font class="cyxy-trs-target" contenteditable="true"> Auucuaug</font></kbd>
                                                                        </strong>
                                                                    </td>
                                                                </tr>
                                                            </tbody>
                                                        </table>
                                                    </div>
                                                    <div id="largeobj_idm140607996152928" class="largeobj-link align_right" style="display: none">
                                                        <a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>
                                                    <div class="tblwrap-foot">
                                                        <div id="FN1">
                                                            <p id="__p23" class="p p-first-last cyxy-trs-source cyxy-trs-source-ted"><sup>a</sup>Library with highest RNA recovery.</p><p id="__p23" class="p p-first-last cyxy-trs-source cyxy-trs-target" contenteditable="true">RNA 提取率最高的图书馆。</p>
                                                        </div>
                                                    </div>
                                                </div>
                                            </div>
                                            <div id="SEC3-2" class="sec">
                                                <h3 id="SEC3-2title" class="cyxy-trs-source cyxy-trs-source-ted">Binding and inhibition screens</h3><h3 id="SEC3-2title" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">捆绑和抑制屏障</h3>
                                                <p id="__p24" class="p p-first cyxy-trs-source cyxy-trs-source-ted">
                                                    The most enriched aptamers for each target enzyme were initially characterized by testing their ability to both bind and inhibit each REase target. Binding affinity and specificity were screened using a qualitative gel shift assay in which 4 nM RNA aptamer was incubated with or without five-fold molar excess REase and then analyzed by electrophoretic gel mobility shift assays in 8% polyacrylamide gels. Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure2A</span>
                                                        </span>
                                                        <span>2A</span>
                                                    </a> shows anti-KpnI aptamers 21–25 as an example. There was evidence of target binding for all aptamers when exposed to cognate REases (Table <a target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid="">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span>
                                                        </span>
                                                        <span>2</span>
                                                    </a>). Aptamer binding was generally specific for the selection target, but a few aptamers (e.g. anti-KpnI aptamers 17 and 27) showed weak non-specific interaction with BamHI.
                                                </p><p id="__p24" class="p p-first cyxy-trs-source cyxy-trs-target" contenteditable="true">对于每个目标酶来说，最富集的适配拥有属性最初是用来测试它们结合和抑制每个 REase 目标的能力。采用定性凝胶转移法筛选4nm RNA 核酸适配体与5倍过量 REase 共孵育的亲和力和特异性，并用8% 聚丙烯酰胺凝胶电泳凝胶迁移试验进行分析。图2A 2A 显示了一个反 kpni 适配子21-25的例子。有证据表明，目标结合时，所有适配体接触到同源 REases (表2)。2).适配子结合对筛选目标具有一定的特异性，但少数适配子(如抗 kpni 适配子17和27)与 BamHI 的非特异性作用较弱。</p>
                                                <!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2">
                                                    <a target="figure" rid-figpopup="F2" rid-ob="ob-F2">
                                                        <!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607994880080" class="figure">
                                                            <img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gkv702fig2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gkv702fig2.jpg" src="http://img.staging.deepbiogroup.com/AptamerImageTotals/822838600001.jpg">
                                                        </div>
                                                    </a><div id="largeobj_idm140607994880080" class="largeobj-link align_right" style="display: none">
                                                        <a target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>

                                                    <div class="icnblk_cntnt" id="lgnd_F2">
                                                        <div>
                                                            <a class="figpopup cyxy-trs-source" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.<font class="cyxy-trs-target"> 图2</font></a>
                                                        </div>
                                                        <!--caption a7--><div class="caption">
                                                            <p id="__p25" class="cyxy-trs-source cyxy-trs-source-ted">Example gels showing results of qualitative screens for aptamer activity. (<strong>A</strong>) Electrophoretic gel mobility shift binding screen. Radiolabeled aptamer candidates (4 nM) were exposed to their corresponding protein targets, in this case KpnI (20 nM; even lanes). (<strong>B</strong>) Restriction inhibition screen. Fluorescent DNA probe (20 nM, see panel <strong>C</strong>) was exposed to REase that had been incubated with a high concentration of the indicated RNA aptamers (40 μM). The examples shown are RNA aptamers against KpnI.</p><p id="__p25" class="cyxy-trs-source cyxy-trs-target" contenteditable="true">例凝胶显示适配体活性的定性筛选结果。(a)电泳凝胶迁移率结合屏。放射标记的候选核酸适配体(4nm)被暴露在相应的蛋白质靶标上，在这种情况下，KpnI (20nm; 偶数通道)。(b)限制性抑制筛。荧光 DNA 探针(20nm，见 c 节)暴露于高浓度 RNA 适配体(40μM)培养的 REase。示例为针对 KpnI 的 RNA 适配子。</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2">
                                                    <h3 class="cyxy-trs-source cyxy-trs-source-ted">Table 2.</h3><h3 class="cyxy-trs-source cyxy-trs-target">表二</h3>
                                                    <!--caption a7--><div class="caption">
                                                        <!--caption a8--><strong class="cyxy-trs-source">Summary of RNA aptamer binding to protein target (4 nM) and enzyme inhibition at an aptamer concentration of 40 μM<font class="cyxy-trs-target" contenteditable="true"> 40μM 适配子浓度下 RNA 适配子与蛋白质靶(4nm)结合及酶抑制的研究进展</font></strong>
                                                    </div>
                                                    <div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607998347952" class="xtable">
                                                        <table frame="hsides" rules="groups" class="rendered small default_table">
                                                            <thead>
                                                                <tr>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">Aptamer<font class="cyxy-trs-target"> 适体</font></th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">Protein target<font class="cyxy-trs-target"> 蛋白质靶</font></th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">BamHI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">PacI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">BamHI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">PacI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></th>
                                                                </tr>
                                                            </thead>
                                                            <tbody>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td align="center" rowspan="1" colspan="1"></td>
                                                                    <td colspan="3" align="center" rowspan="1">
                                                                        <em class="cyxy-trs-source">protein binding<font class="cyxy-trs-target" contenteditable="true"> 蛋白质结合</font></em>
                                                                        <sup>a</sup>
                                                                    </td>
                                                                    <td colspan="3" align="center" rowspan="1">
                                                                        <em class="cyxy-trs-source">enzyme inhibition<font class="cyxy-trs-target"> 酶抑制</font></em>
                                                                        <sup>a</sup>
                                                                    </td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">1/9/10<font class="cyxy-trs-target" contenteditable="true"> 1910年1月9日</font></td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">BamHI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">2</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">BamHI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">3/7</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">BamHI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">4</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">BamHI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">5/13</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">BamHI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">6/12<font class="cyxy-trs-target" contenteditable="true"> 6月12日</font></td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">BamHI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">8</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">BamHI<font class="cyxy-trs-target" contenteditable="true"> 女名女子名</font></td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">15</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">16/23/29</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">17/22/28</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</td>
                                                                    <td align="center" rowspan="1" colspan="1">±</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">18/21/26</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">±</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">19</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">20</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">24</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">±</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">25</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">27</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</td>
                                                                    <td align="center" rowspan="1" colspan="1">±</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">30</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">KpnI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">38/33</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">PacI</td>
                                                                    <td align="center" rowspan="1" colspan="1">±</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">±</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">39/34/36</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">PacI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">±</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">40/51</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">PacI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">±</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">41/32</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">PacI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">±</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">44</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">PacI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">±</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">45</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">PacI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">±</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">46/50</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">PacI</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">+</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">52</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">anti-p50</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">31</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">T2</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                    <td align="center" rowspan="1" colspan="1">−</td>
                                                                </tr>
                                                            </tbody>
                                                        </table>
                                                    </div>
                                                    <div id="largeobj_idm140607998347952" class="largeobj-link align_right" style="display: none">
                                                        <a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>
                                                    <div class="tblwrap-foot">
                                                        <div id="FN2">
                                                            <p id="__p26" class="p p-first-last cyxy-trs-source"><sup>a</sup>Binding/inhibition, (+); partial binding/inhibition, (±); no binding/inhibition (-).</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <p id="__p27" class="p p-last cyxy-trs-source">
                                                    Initial qualitative REase inhibition screening was performed by testing the capacity of a single high concentration of each aptamer to inhibit BamHI, KpnI and PacI cleavage. A 321-bp fluorescent duplex DNA probe carrying single recognition sequences for each REase was designed for this purpose Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure2C.</span>
                                                        </span>
                                                        <span>2C</span>
                                                    </a>. Example data are shown in Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure2B.</span>
                                                        </span>
                                                        <span>2B</span>
                                                    </a>. To assess specificity, all aptamers were screened for their ability to inhibit all three of the target enzymes. Interestingly, only anti-KpnI RNA aptamers showed both selective binding and restriction inhibition (Table <a target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid="">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">(Table2</span>
                                                        </span>
                                                        <span>2</span>
                                                    </a>).
                                                </p>
                                            </div>
                                            <div id="SEC3-3" class="sec">
                                                <h3 id="SEC3-3title" class="cyxy-trs-source">Quantitating KpnI binding by RNA aptamers</h3>
                                                <p id="__p28" class="p p-first cyxy-trs-source">
                                                    To evaluate quantitatively, the affinities of four representative anti-KpnI aptamers, aptamer 20, 24, 29 and 30, (Table <a target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid="">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">(Table3)</span>
                                                        </span>
                                                        <span>3</span>
                                                    </a>) we used an electrophoretic gel mobility shift assay where radiolabeled RNA aptamers (15 nM) were titrated with increasing concentrations of KpnI (aptamer 20 example shown in Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure3).</span>
                                                        </span>
                                                        <span>3</span>
                                                    </a>). Gel images were quantitated and equilibrium dissociation constant (<em>K</em>
                                                    <sub>d</sub>) and uncertainty values estimated by fitting as described in ‘Materials and Methods’ section. Summary data are shown in Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure4</span>
                                                        </span>
                                                        <span>4</span>
                                                    </a> and Table&nbsp;<a target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid="">
                                                        <span>4</span>
                                                    </a>.
                                                </p>
                                                <!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3">
                                                    <a target="figure" rid-figpopup="F3" rid-ob="ob-F3">
                                                        <!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607998216624" class="figure">
                                                            <img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gkv702fig3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gkv702fig3.jpg" src="http://img.staging.deepbiogroup.com/AptamerImageTotals/822838600002.jpg">
                                                        </div>
                                                    </a><div id="largeobj_idm140607998216624" class="largeobj-link align_right" style="display: none">
                                                        <a target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>

                                                    <div class="icnblk_cntnt" id="lgnd_F3">
                                                        <div>
                                                            <a class="figpopup cyxy-trs-source" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3.</a>
                                                        </div>
                                                        <!--caption a7--><div class="caption">
                                                            <p id="__p29" class="cyxy-trs-source">Example of anti-KpnI aptamer 20 binding to KpnI as measured by quantitative electrophoretic gel mobility shift assay. A total of 13 nM radiolabeled RNA aptamer titrated with increasing concentrations of KpnI as indicated.</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4">
                                                    <a target="figure" rid-figpopup="F4" rid-ob="ob-F4">
                                                        <!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607990001312" class="figure">
                                                            <img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gkv702fig4.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gkv702fig4.jpg" src="http://img.staging.deepbiogroup.com/AptamerImageTotals/822838600003.jpg">
                                                        </div>
                                                    </a><div id="largeobj_idm140607990001312" class="largeobj-link align_right" style="display: none">
                                                        <a target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>

                                                    <div class="icnblk_cntnt" id="lgnd_F4">
                                                        <div>
                                                            <a class="figpopup cyxy-trs-source" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4.</a>
                                                        </div>
                                                        <!--caption a7--><div class="caption">
                                                            <p id="__p30" class="cyxy-trs-source">Quantitation of KpnI binding affinity by electrophoretic gel mobility shift assay for (<strong>A</strong>) aptamer 20; (<strong>B</strong>) aptamer 24; (<strong>C</strong>) aptamer 29; and (<strong>D</strong>) aptamer 30. Replicate data are shown as black and gray points.</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3">
                                                    <h3 class="cyxy-trs-source">Table 3.</h3>
                                                    <!--caption a7--><div class="caption">
                                                        <!--caption a8--><strong class="cyxy-trs-source cyxy-trs-source">Sequences of representative anti-KpnI aptamers<sup>a</sup> and aptamer 20 mutants</strong>
                                                    </div>
                                                    <div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607999528496" class="xtable">
                                                        <table frame="hsides" rules="groups" class="rendered small default_table">
                                                            <tbody>
                                                                <tr>
                                                                    <td rowspan="1" colspan="1">
                                                                        <div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607991419280" class="figure">
                                                                            <img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gkv702ufig1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gkv702ufig1.jpg" src="">
                                                                        </div>
                                                                        <div id="largeobj_idm140607991419280" class="largeobj-link align_right" style="display: none">
                                                                            <a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                                        </div>
                                                                    </td>
                                                                </tr>
                                                            </tbody>
                                                        </table>
                                                    </div>
                                                    <div id="largeobj_idm140607999528496" class="largeobj-link align_right" style="display: none">
                                                        <a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>
                                                </div>
                                                <!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4">
                                                    <h3 class="cyxy-trs-source">Table 4.</h3>
                                                    <!--caption a7--><div class="caption">
                                                        <!--caption a8--><strong class="cyxy-trs-source cyxy-trs-source cyxy-trs-source cyxy-trs-source cyxy-trs-source">
                                                            Apparent equilibrium dissociation constant (<em>K</em>
                                                            <sub>d</sub>), Hill coefficient (<em>h</em>), maximum fraction of bound RNA (<em>f</em>
                                                            <sub class="cyxy-trs-source">max</sub>) and minimum fraction of bound RNA (<em>f</em>
                                                            <sub class="cyxy-trs-source">min</sub>) for selected aptamers binding to KpnI
                                                        </strong>
                                                    </div>
                                                    <div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607994670464" class="xtable">
                                                        <table frame="hsides" rules="groups" class="rendered small default_table">
                                                            <thead>
                                                                <tr>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">
                                                                        Aptamer<sup>a</sup>
                                                                    </th>
                                                                    <th align="center" rowspan="1" colspan="1">
                                                                        <em>
                                                                            K<sub>d</sub>
                                                                        </em>
                                                                    </th>
                                                                    <th align="center" rowspan="1" colspan="1">
                                                                        <em>h</em>
                                                                    </th>
                                                                    <th align="center" rowspan="1" colspan="1">
                                                                        <em>f</em>
                                                                        <sub class="cyxy-trs-source">max</sub>
                                                                    </th>
                                                                    <th align="center" rowspan="1" colspan="1">
                                                                        <em>f</em>
                                                                        <sub class="cyxy-trs-source">min</sub>
                                                                    </th>
                                                                </tr>
                                                            </thead>
                                                            <tbody>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">20</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">11.6 ± 2.3 nM</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">1.68 ± 0.48</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">0.97 ± 0.03</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">0.13 ± 0.06</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">24</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">28.7 ± 5.2 nM</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">2.83 ± 0.95</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">0.94 ± 0.04</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">0.20 ± 0.02</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">29</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">419 ± 46.6 nM</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">1.17 ± 1.18</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">1.00 ±0.07</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">0.42 ±0.13</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">30</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">18.6 ± 4.5 nM</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">2.45 ± 0.85</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">0.95 ± 0.02</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">0.29 ± 0.02</td>
                                                                </tr>
                                                            </tbody>
                                                        </table>
                                                    </div>
                                                    <div id="largeobj_idm140607994670464" class="largeobj-link align_right" style="display: none">
                                                        <a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>
                                                    <div class="tblwrap-foot">
                                                        <div id="FN3">
                                                            <p id="__p31" class="p p-first-last cyxy-trs-source"><sup>a</sup>Parameters reported reflect the mean and standard deviation of 10 000 fits to experimental data (at least two replicates) with added Gaussian noise (see ‘Materials and Methods’ section).</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <p id="__p32" class="p cyxy-trs-source">
                                                    Similar aptamers 20 and 30 displayed tight binding to KpnI, with <em>K</em>
                                                    <sub>d</sub> values of 11.6 ± 2.3 nM and 18.6 ± 4.5 nM, respectively (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4A,</span>
                                                        </span>
                                                        <span>4A</span>
                                                    </a>, <a target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -0.5em;">,D</span>
                                                        </span>
                                                        <span>D</span>
                                                    </a> and Table <a target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid="">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -2.5em;">Table4).</span>
                                                        </span>
                                                        <span>4</span>
                                                    </a>). Aptamers 24 and 29 displayed <em>K</em>
                                                    <sub>d</sub> values ∼2- and ∼25-fold weaker specificity, respectively (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4B</span>
                                                        </span>
                                                        <span>4B</span>
                                                    </a> and&nbsp;<a target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4">
                                                        <span>C</span>
                                                    </a>). As shown in Table&nbsp;<a target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid="">
                                                        <span>4</span>
                                                    </a>, fit values of the Hill coefficient were greater than unity, typically suggestive of positive cooperativity in multi-ligand binding reactions. The data in Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure3,</span>
                                                        </span>
                                                        <span>3</span>
                                                    </a>, lanes 10–15 indeed showed some evidence of multiple KpnI proteins bound per aptamer at very high protein concentration, perhaps contributing to Hill coefficients. However, only a single (apparently 1:1) aptamer monomer/KpnI dimer complex was detected when a fixed concentration of KpnI protein (50 nM) was incubated with a wide range of aptamer concentrations (see example in Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure5).</span>
                                                        </span>
                                                        <span>5</span>
                                                    </a>). There is only a small trace of higher molecular weight complex in the presence of high excess anti-KpnI aptamer (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5,</span>
                                                        </span>
                                                        <span>5</span>
                                                    </a>, lanes 13–15). This suggests that single anti-KpnI RNA aptamers bind KpnI protein dimers.
                                                </p>
                                                <!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5">
                                                    <a target="figure" rid-figpopup="F5" rid-ob="ob-F5">
                                                        <!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607991813264" class="figure">
                                                            <img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gkv702fig5.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gkv702fig5.jpg" src="http://img.staging.deepbiogroup.com/AptamerImageTotals/822838600004.jpg">
                                                        </div>
                                                    </a><div id="largeobj_idm140607991813264" class="largeobj-link align_right" style="display: none">
                                                        <a target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>

                                                    <div class="icnblk_cntnt" id="lgnd_F5">
                                                        <div>
                                                            <a class="figpopup cyxy-trs-source" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5.</a>
                                                        </div>
                                                        <!--caption a7--><div class="caption">
                                                            <p id="__p33" class="cyxy-trs-source">Titration of 50 nM KpnI with high affinity radiolabeled aptamer 20 in the concentration range 100 pM–500 nM.</p>
                                                        </div>
                                                    </div>
                                                </div>
                                            </div>
                                            <div id="SEC3-4" class="sec">
                                                <h3 id="SEC3-4title" class="cyxy-trs-source">Quantitating KpnI inhibition by RNA aptamers</h3>
                                                <p id="__p34" class="p p-first cyxy-trs-source">
                                                    Quantitation of enzyme inhibition was performed by titrating, in triplicate, 10 pM–100 μM final concentrations of 15 anti-KpnI aptamers in cleavage reactions containing 20 nM fluorescent DNA probe and 0.2 U/μl KpnI in standard KpnI digestion buffer. The aptamer concentration to inhibit 50% of probe cleavage under these conditions (IC<sub>50</sub>) was determined by curve-fitting and estimations of uncertainty by Monte Carlo simulations, see ‘Materials and Methods’ section. IC<sub>50</sub> values ranged between 13 nM and 18 μM depending on the aptamer. Representative examples are shown in Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure6</span>
                                                        </span>
                                                        <span>6</span>
                                                    </a> and Table <a target="table" class="fig-table-link figpopup" rid-figpopup="tbl5" rid-ob="ob-tbl5" co-legend-rid="">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -2.5em;">Table5.</span>
                                                        </span>
                                                        <span>5</span>
                                                    </a>. Similar anti-KpnI aptamers 20 and 30 were the most powerful selective inhibitors, with IC<sub>50</sub> of values 36.1 ± 3.9 nM and 23.1 ± 0.8 nM, respectively, in this assay (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6A</span>
                                                        </span>
                                                        <span>6A</span>
                                                    </a> and&nbsp;<a target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6">
                                                        <span>D</span>
                                                    </a>). Interestingly, aptamer 24 shown in Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure4B</span>
                                                        </span>
                                                        <span>4B</span>
                                                    </a> to have tight binding affinity, was a poor KpnI inhibitor (IC<sub>50</sub> ∼ 18 ± 5.4 μM) (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6B).</span>
                                                        </span>
                                                        <span>6B</span>
                                                    </a>). Aptamer 29 was relatively weaker in both binding and inhibition (Figures <a target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -4em;">(Figures4C</span>
                                                        </span>
                                                        <span>4C</span>
                                                    </a> and&nbsp;<a target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6">
                                                        <span>6C</span>
                                                    </a>).
                                                </p>
                                                <!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F6" co-legend-rid="lgnd_F6">
                                                    <a target="figure" rid-figpopup="F6" rid-ob="ob-F6">
                                                        <!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607990866704" class="figure">
                                                            <img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gkv702fig6.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gkv702fig6.jpg" src="http://img.staging.deepbiogroup.com/AptamerImageTotals/822838600005.jpg">
                                                        </div>
                                                    </a><div id="largeobj_idm140607990866704" class="largeobj-link align_right" style="display: none">
                                                        <a target="figure" rid-figpopup="F6" rid-ob="ob-F6"></a><a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>

                                                    <div class="icnblk_cntnt" id="lgnd_F6">
                                                        <div>
                                                            <a class="figpopup cyxy-trs-source" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6.</a>
                                                        </div>
                                                        <!--caption a7--><div class="caption">
                                                            <p id="__p35" class="cyxy-trs-source">KpnI inhibition by anti-KpnI RNA aptamers. KpnI in the presence of 20 nM fluorescent DNA probe was incubated with anti-KpnI aptamers in the concentration range 10 pM–100 μM. (<strong>A</strong>) Aptamer 20; (<strong>B</strong>) aptamer 24; (<strong>C</strong>) aptamer 29; (<strong>D</strong>) aptamer 30.</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl5">
                                                    <h3 class="cyxy-trs-source">Table 5.</h3>
                                                    <!--caption a7--><div class="caption">
                                                        <!--caption a8--><strong class="cyxy-trs-source cyxy-trs-source cyxy-trs-source">Relative half maximal inhibitory concentration (IC<sub>50</sub>) and slope factor (<em>s</em>) for selected aptamers inhibiting KpnI</strong>
                                                    </div>
                                                    <div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607995031520" class="xtable">
                                                        <table frame="hsides" rules="groups" class="rendered small default_table">
                                                            <thead>
                                                                <tr>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">
                                                                        Aptamer<sup>a</sup>
                                                                    </th>
                                                                    <th align="center" rowspan="1" colspan="1">
                                                                        <em>s</em>
                                                                    </th>
                                                                    <th align="center" rowspan="1" colspan="1" class="cyxy-trs-source">
                                                                        IC<sub>50</sub>
                                                                    </th>
                                                                </tr>
                                                            </thead>
                                                            <tbody>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">20</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">1.45 ± 0.14</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">36.1 ± 3.9 nM</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">24</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">1.72 ± 2.09</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">18.0 ± 5.4 μM</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">29</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">1.92 ± 0.16</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">153 ± 11 nM</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1">30</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">1.43 ± 0.07</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">23.1 ± 0.8 nM</td>
                                                                </tr>
                                                                <tr>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">20s</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">1.77 ± 0.25</td>
                                                                    <td align="center" rowspan="1" colspan="1" class="cyxy-trs-source">13.0 ± 1.5 nM</td>
                                                                </tr>
                                                            </tbody>
                                                        </table>
                                                    </div>
                                                    <div id="largeobj_idm140607995031520" class="largeobj-link align_right" style="display: none">
                                                        <a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>
                                                    <div class="tblwrap-foot">
                                                        <div id="FN4">
                                                            <p id="__p36" class="p p-first-last cyxy-trs-source"><sup>a</sup>Parameters reported with uncertainty reflect the mean and standard deviation of 10 000 fits to experimental data (at least three replicates) with added Gaussian noise (see ‘Materials and Methods’ section).</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <p id="__p37" class="p cyxy-trs-source">
                                                    We hypothesized that the selected anti-KpnI RNA aptamers function as competitive inhibitors of KpnI. This was tested by monitoring the ability of anti-KpnI RNA aptamers to compete with the fluorescent DNA probe for binding to KpnI protein. The results of two examples, aptamer 20 and 24 are shown in Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure7.</span>
                                                        </span>
                                                        <span>7</span>
                                                    </a>. Under these binding conditions commercial KpnI preparations show multiple shifted DNA complexes due to partial protein oxidation (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7A,</span>
                                                        </span>
                                                        <span>7A</span>
                                                    </a>, lanes 2–6 and Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure7B,</span>
                                                        </span>
                                                        <span>7B</span>
                                                    </a>, lanes 2–12). Interestingly, aptamer 20 (strong binding and strong inhibition) acts as a competitor, as expected, (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7A),</span>
                                                        </span>
                                                        <span>7A</span>
                                                    </a>), while aptamer 24 (strong binding but weak inhibition) does not (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure7B).</span>
                                                        </span>
                                                        <span>7B</span>
                                                    </a>). This suggests that aptamer 24 may bind KpnI without occluding its active site.
                                                </p>
                                                <!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F7" co-legend-rid="lgnd_F7">
                                                    <a target="figure" rid-figpopup="F7" rid-ob="ob-F7">
                                                        <!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607995421104" class="figure">
                                                            <img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gkv702fig7.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gkv702fig7.jpg" src="http://img.staging.deepbiogroup.com/AptamerImageTotals/822838600006.jpg">
                                                        </div>
                                                    </a><div id="largeobj_idm140607995421104" class="largeobj-link align_right" style="display: none">
                                                        <a target="figure" rid-figpopup="F7" rid-ob="ob-F7"></a><a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>

                                                    <div class="icnblk_cntnt" id="lgnd_F7">
                                                        <div>
                                                            <a class="figpopup cyxy-trs-source" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Figure 7.</a>
                                                        </div>
                                                        <!--caption a7--><div class="caption">
                                                            <p id="__p38" class="cyxy-trs-source">Competition gel shift assay showing inhibition of KpnI binding to fluorescent DNA probe in the presence of 100 pM–10 μM anti-KpnI RNA aptamers. (<strong>A</strong>) Aptamer 20, (<strong>B</strong>) aptamer 24.</p>
                                                        </div>
                                                    </div>
                                                </div>
                                            </div>
                                            <div id="SEC3-5" class="sec">
                                                <h3 id="SEC3-5title" class="cyxy-trs-source">Specificity of REase inhibition</h3>
                                                <p id="__p39" class="p p-first-last cyxy-trs-source">
                                                    We initially determined that even 40 μM anti-KpnI RNA aptamers did not inhibit DNA cleavage by BamHI or PacI (Table <a target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid="">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span>
                                                        </span>
                                                        <span>2</span>
                                                    </a>). We considered a more stringent test of specificity by challenging the KpnI isoschizomer Acc65I (identical DNA target sequence specificity) with aptamer 20. Remarkably, this aptamer is completely specific for KpnI (Supplementary Figure S1). This result suggests that the DNA sequence mimicry of anti-KpnI aptamer 20 must include an element of protein interaction outside the minimal DNA recognition interface.
                                                </p>
                                            </div>
                                            <div id="SEC3-6" class="sec">
                                                <h3 id="SEC3-6title" class="cyxy-trs-source">Anti-KpnI RNA aptamer secondary structures</h3>
                                                <p id="__p40" class="p p-first cyxy-trs-source">
                                                    The RNA aptamer library employed in these selections used as a scaffold a compact hairpin inspired by the high-affinity anti-NF-κB p50 RNA aptamer (<a rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922024">12</a>,<a rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921997">17</a>,<a rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922013">23</a>). To test if this structural framework was conserved after selection, secondary structure predictions for anti-KpnI aptamers 20, 24, 29 and 30 were generated (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8B</span>
                                                        </span>
                                                        <span>8B</span>
                                                    </a> and Supplementary Figure S2). Indeed, predicted lowest-energy folds for all aptamers retain the hairpin approximate, each with bulged and unpaired nucleotides derived from random regions and the invariant tetraloop and paired stem elements approximately as expected. We note that nucleotides predicted to be in unpaired internal loops often are discovered to be participating in complex non-canonical interactions (<a rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921990">23</a>).
                                                </p>
                                                <!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F8" co-legend-rid="lgnd_F8">
                                                    <a target="figure" rid-figpopup="F8" rid-ob="ob-F8">
                                                        <!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140607995230928" class="figure">
                                                            <img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gkv702fig8.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gkv702fig8.jpg" src="http://img.staging.deepbiogroup.com/AptamerImageTotals/822838600007.jpg">
                                                        </div>
                                                    </a><div id="largeobj_idm140607995230928" class="largeobj-link align_right" style="display: none">
                                                        <a target="figure" rid-figpopup="F8" rid-ob="ob-F8"></a><a target="object" class="cyxy-trs-source">Open in a separate window<font class="cyxy-trs-target"> 在单独的窗口中打开</font></a>
                                                    </div>

                                                    <div class="icnblk_cntnt" id="lgnd_F8">
                                                        <div>
                                                            <a class="figpopup cyxy-trs-source" target="figure" rid-figpopup="F8" rid-ob="ob-F8">Figure 8.</a>
                                                        </div>
                                                        <!--caption a7--><div class="caption">
                                                            <p id="__p41" class="cyxy-trs-source">(<strong>A</strong>) In-line probing (<a rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922018">36</a>)&nbsp;of high-affinity anti-KpnI aptamers 20, 24, 29, 30. (UN: unmodified samples (lane 1, 6, 11 and 16); T1: RNAse T1 digestions (lane 2, 7, 12 and 17); OH: alkaline hydrolysis reactions (lane 3, 8, 13 and 18); IN: In-line attack reactions after 24 h (lane 4, 9, 14 and 19) and 48 h (lane 5, 10, 15 and 20). (<strong>B</strong>) Predicted secondary structure of aptamer 20 and validation by in-line probing shown as color-coded heat map (red, high in-line attack rate; white, intermediate in-line attack rate; blue, low in-line attack rate). Black box highlights the nucleotides that were randomized in the original library.</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <p id="__p42" class="cyxy-trs-source">
                                                    The validity of these predicted secondary structures was tested by in-line probing analysis (<a rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922005">27</a>). This technique exploits the intrinsic chemical instability of RNA due to spontaneous nucleophilic attack of a 2′ hydroxyl oxygen on the adjacent phosphorous center. This spontaneous attack is accelerated when the nucleophile is ‘in-line’ with the adjacent phosphorus center and the 5′ oxygen-leaving group of the linkage, a scenario that is suppressed in conventional Watson–Crick base-paired stems (<a rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922028">27</a>–<a rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921982">29</a>,<a rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921983">36</a>). In-line probing reactions for anti-KpnI aptamers 20, 24, 29 and 30 are shown in Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure8A.</span>
                                                        </span>
                                                        <span>8A</span>
                                                    </a>. We compared uniform alkali cleavage (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8A,</span>
                                                        </span>
                                                        <span>8A</span>
                                                    </a>, lanes 3, 8, 13, 18) to in-line attack lanes (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8A,</span>
                                                        </span>
                                                        <span>8A</span>
                                                    </a>, lanes 4/5, 9/10, 14/15, 19/20, respectively) and a heat map was generated showing low to high in-line attack rates at base pair resolution. An example for aptamer 20 is shown in Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure8B</span>
                                                        </span>
                                                        <span>8B</span>
                                                    </a> and the remaining cases in Supplementary Figure S2. As shown in Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">Figure8B,</span>
                                                        </span>
                                                        <span>8B</span>
                                                    </a>, nucleotides predicted to be base-paired tend to show suppressed in-line attack rate, as expected (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8B,</span>
                                                        </span>
                                                        <span>8B</span>
                                                    </a>, G1–A6, G34–C39, A41–C47 and G55–U58) corresponding to heat-map scores in shades of blue. Regions of flexibility or structures where the 2′ OH group more frequently samples an in-line conformation have higher rates of self-cleavage, as predicted (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8B,</span>
                                                        </span>
                                                        <span>8B</span>
                                                    </a>, A22–G26, A30–C33, A48–G50 and A52–C54) and shown in shades of red in heat map (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8B</span>
                                                        </span>
                                                        <span>8B</span>
                                                    </a> and Supplementary Figure S2). We note that the RNA secondary structure prediction algorithm used here does not take into account the relative stability of A–G mispairs (<a rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922012">30</a>). There are potential pairs of this kind (for example A25–G36 and A6–G53) and such pairs might help explain differential in-line attack profiles on opposite strands of a potential double helix.
                                                </p>
                                                <p id="__p43" class="p p-last cyxy-trs-source">
                                                    Based on the secondary structure prediction and in-line probing data for anti-KpnI aptamer 20 (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8B),</span>
                                                        </span>
                                                        <span>8B</span>
                                                    </a>), we tested if a minimal stem-loop structure from positions 12–46 (aptamer 20s) was sufficient for KpnI inhibition (Table <a target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid="">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">(Table3).</span>
                                                        </span>
                                                        <span>3</span>
                                                    </a>). Indeed, this 38-nt RNA aptamer is a potent KpnI inhibitor (Supplementary Figure S3).
                                                </p>
                                            </div>
                                            <div id="SEC3-7" class="sec">
                                                <h3 id="SEC3-7title" class="cyxy-trs-source">Further study of minimal recognition sequence</h3>
                                                <p id="__p44" class="p p-first-last cyxy-trs-source">
                                                    To further explore minimal recognition sequences of the anti-KpnI aptamers we created four mutants of aptamer 20. A scrambled version of aptamer 20 was created (20_scram) in which the nucleotide sequences of the random regions (N10 and N9) were randomized. A 26-nt truncated variant of aptamer 20 (20_26 nt) presented a shorter version of the putative recognition hairpin. A variant of aptamer 20 (20_AAA) was mutated in a distinctive stretch of four adenosine residues. Finally, a variant aptamer (20_downst) contains mutations within the N9 random region downstream of the tetraloop (Table <a target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid="">
                                                        <span style="position: relative;text-decoration:none;">
                                                            ​<span class="figpopup-sensitive-area" style="left: -3em;">(Table3).</span>
                                                        </span>
                                                        <span>3</span>
                                                    </a>). Secondary structure predictions for these variant aptamers suggest that only 20_AAA and 20_downs conserve aspects of aptamer 20 structure, while 20_26 nt and 20_scram differ greatly (Supplementary Figure S4). Importantly, digestion inhibition experiments showed that all four of these mutant aptamers lost inhibitory activity (IC<sub>50</sub> values in the hundreds of micromolar; Supplementary Figure S5.) These results confirm that KpnI binding involves nucleotides mapping to the random region of the stem-loop, as expected.
                                                </p>
                                            </div>
                                            <div id="SEC3-8" class="sec sec-last">
                                                <h3 id="SEC3-8title" class="cyxy-trs-source">CD spectroscopy</h3>
                                                <p id="__p45" class="p p-first-last cyxy-trs-source">In an attempt to evaluate the hypothesis that restriction inhibition involves DNA mimicry by nucleotides within the stem-loop domain of aptamer 20, we compared CD spectra of aptamers 20, 20s, 20_scramble, 20s_scramble, 20_26 nt, 20_AAA, 20_downs, with known DNA mimic anti-NF-κB p50 serving as control. As shown in Supplementary Figures S6 and S7, CD spectra for all of these molecules are dominated by the signal of the overall RNA stem-loop structure [265 nm (+), 240 nm (−), 220 nm (+) and 210 nm (−) (<a rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469921992">31</a>–<a rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922025">33</a>)]. In addition, difference spectra were not interpretable. This result indicates that CD spectroscopy is not well-suited to drawing conclusions about potential B-DNA mimicry within a small sub-domain of an RNA aptamer.</p>
                                            </div>
                                        </div>
                                        <div id="SEC4" class="tsec sec">
                                            <h2 class="head no_bottom_margin cyxy-trs-source" id="SEC4title" style="text-transform: uppercase;">DISCUSSION</h2>
                                            <p id="__p46" class="p p-first cyxy-trs-source">
                                                Here we report the <em>in</em>
                                                <em>vitro</em> selection and characterization of RNA aptamers using REases BamHI, KpnI and PacI as targets. We identify RNA aptamers selected against KpnI that selectively bind (<em>K</em>
                                                <sub>d</sub> ∼12–30 nM) and inhibit (IC<sub>50</sub> ∼20–150 nM) this REase with high specificity. Evaluation of four representative anti-KpnI aptamers (20, 24, 29 and 30) by reverse titration and by competition studies suggests a 1 aptamer:1 KpnI homodimer binding stoichiometry. Binding appears to be at or near the KpnI DNA binding site, competing with DNA binding. Binding selectivity can be exquisite: mimicry of the KpnI DNA binding site by aptamer 20 does not confer inhibition of KpnI isoschizomer Acc65I. This suggests that aptamer 20 contacts KpnI surfaces beyond the minimal DNA recognition interface.
                                            </p>
                                            <p id="__p47" class="cyxy-trs-source">Secondary structure prediction, in-line probing experiments and CD spectra suggest that the selected aptamers evolved on the approximate intended hairpin. KpnI inhibition by a minimal version of aptamer 20 (aptamer 20s) confirms this prediction. Interestingly, none of the inhibitory anti-KpnI aptamers carries any obvious homology to the duplex DNA binding site 5′-GGTACC. This is not unexpected in that the previously-studied high-affinity anti-NF-κB p50 RNA aptamer also does not bear obvious primary sequence homology to the DNA sequence it mimics (<a rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922004">12</a>,<a rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922011">17</a>,<a rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922022">23</a>).</p>
                                            <p id="__p48" class="cyxy-trs-source">
                                                From the study of several aptamer 20 mutants we deduce that nucleotides C12–U17 of the sequence 5′-CAUGAU are crucial for promoting the suggested hairpin structure necessary for target binding. Disruption of this sequence inactivates restriction inhibition (Supplementary Figures S3 and S5). We also found that disruption of either the distinctive A<sub>4</sub> sequence within the N10 domain or the N9 sequence downstream from the tetraloop inactivated function even when an overall hairpin structure was roughly conserved (Figure <a target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8">
                                                    <span style="position: relative;text-decoration:none;">
                                                        ​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure8B</span>
                                                    </span>
                                                    <span>8B</span>
                                                </a> and Supplementary Figure S4). This confirms that the minimal structure within the 31-nt 20s depends on the selected random region, as expected. Future studies could vary the character of the tetraloop, a feature found important in the case of the anti-NF-κB RNA aptamer, where loop optimization afforded a ∼20-fold improvement in activity (<a rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922001">16</a>).
                                            </p>
                                            <p id="__p49" class="cyxy-trs-source">Regrettably, there are no high-resolution structural data for KpnI, a member of the HNH superfamily. Homology analysis suggests that amino acid residues D148, H149 and Q175 compose the catalytic metal-binding active site of KpnI (<a rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922006">34</a>). Furthermore, CD spectra of aptamer 20 and several aptamer 20 variants were substantially similar and comparable to that of the anti-NF-κB RNA aptamer structure that had been the basis for the design scaffold&nbsp;Supplementary Figures S6 and S7). This result confirms that the anti-KpnI aptamers evolved within the intended hairpin scaffold and this structure dominates the CD spectrum, preventing conclusions to be drawn about details of B-DNA mimicry within a small region of the structure. We predict that eventual high-resolution structure determination for complexes of KpnI and anti-KpnI RNA aptamers will reveal RNA binding that obscures this active site. Based on our previous studies of the anti-NF-κB RNA aptamer we hypothesize that the anti-KpnI aptamers reported here have evolved as structural mimics of the DNA major groove recognized by KpnI.</p>
                                            <p id="__p50" class="cyxy-trs-source">Our prior selections of RNA aptamers against NF-κB family members (<a rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922003">17</a>,<a rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469922026">35</a>) suggest that DNA mimics may be the primary products for DNA binding protein targets. The anti-NF-κB p50 aptamer studied in greatest detail was characterized by a predicted hairpin structure with asymmetric internal loop. There was no obvious sequence similarity to NF-κB binding sites in DNA and no hint from chemical or enzymatic probing as to the mechanism of protein recognition. Only with the solution of X-ray co-crystals of the aptamer and its protein target and the determination of the aptamer solution structure by nuclear magnetic resonance did it become clear that the aptamer had been selected for elegant structural mimicry of the DNA major groove, allowing the RNA to engage precisely the same face of the transcription factor that normally engages DNA. We therefore hypothesize that RNA aptamers selected against DNA binding proteins are likely to evolve interfaces based on DNA mimicry. It remains to be seen through high-resolution structural studies if that is the mechanism of the anti-KpnI RNA aptamers described here. By designing the present selections in the context of an RNA structural scaffold based on the previously-studied anti-NF-κB RNA aptamer, our ultimate goal is to test the hypothesis that DNA groove mimicry is a general mechanism for RNA binding to DNA binding proteins. In this sense, the new anti-KpnI RNA aptamers described here are now candidates for high-resolution structural study to judge the validity of this hypothesis.</p>
                                            <p id="__p51" class="cyxy-trs-source">It is intriguing that RNA aptamers selected against BamHI and PacI showed binding activity without enzyme inhibition. We hypothesize that three considerations are at play. First, these enzymes may have cationic surfaces that elicit RNA ligands lacking the ability to inhibit enzyme activity. Second, we found it necessary to identify different stringent salt conditions for selections with each enzyme. This means that the selection conditions for BamHI (very high salt in the presence of Mg<sup>2+)</sup> and for PacI (very low salt in the presence of Ca<sup>2+</sup>) are least like the actual conditions for enzyme activity. Finally, we note that BamHI selections employed catalytically inactive E111A mutant protein. It is possible that the selected aptamers do not bind as well to catalytically-active wild-type protein.</p>
                                            <p id="__p52" class="cyxy-trs-source">When KpnI inhibitory RNA aptamers were discovered, we considered the possibility that DNA mimicry might be so effective that RNA cleavage by KpnI could result. Such RNA cleavage was not observed among the studied aptamers (data not shown). However, it is possible that RNA cleavage might rapidly inactivate such aptamers so they do not function as competitive inhibitors.</p>
                                            <p id="__p53" class="p p-last cyxy-trs-source">
                                                To our knowledge, the anti-KpnI RNA aptamers reported here are the first known selective restriction enzyme inhibitors. Current approaches to REase inhibition are limited to irreversible heat denaturation or Mg<sup>2+</sup> chelation. Neither approach is selective, reversible or applicable <em>in</em>
                                                <em>vivo</em>. Anti-REase RNA aptamers could prove valuable in studies of restriction enzyme mechanism. Moreover, these aptamers provide new examples in the growing list of natural and artificial RNAs that bind DNA-binding proteins by DNA mimicry. As such, they provide the raw material for future structural studies to understand in detail this DNA mimicry and to deduce principles by which an RNA scaffold might be designed to inhibit transcription factors with known DNA binding specificities.
                                            </p>
                                        </div>
                                        <div id="__sec1" class="tsec sec">
                                            <a id="supplementary-material-sec"></a>
                                            <h2 class="head no_bottom_margin cyxy-trs-source" id="__sec1title" style="text-transform: uppercase;">Supplementary Material</h2>
                                            <!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

                                                <!--caption a9--><strong class="cyxy-trs-source">SUPPLEMENTARY DATA:</strong>

                                                <div class="sup-box half_rhythm" id="idm140608037991344">
                                                    <a data-ga-action="click_feat_suppl" class="cyxy-trs-source">Click here to view.</a>
                                                </div>

                                            </div>
                                        </div>
                                        <div id="idm140607995963824" class="tsec sec">
                                            <h2 class="head no_bottom_margin cyxy-trs-source" id="idm140607995963824title" style="text-transform: uppercase;">Acknowledgments</h2>
                                            <div class="sec">
                                                <p id="__p54" class="cyxy-trs-source">
                                                    The authors acknowledge Drs. Geoff Wilson and Dan Heiter (New England Biolabs) for generously provided purified REases. We greatly appreciate the expertise and assistance Dr. Justin Peters in data fitting, figure preparation and Monte Carlo simulations to compute the uncertainty of <em>K</em>
                                                    <sub>d</sub> and IC<sub>50</sub> values.
                                                </p>
                                            </div>
                                        </div>
                                        <div id="SEC5" class="tsec bk-sec">
                                            <h2 class="head no_bottom_margin cyxy-trs-source" id="SEC5title" style="text-transform: uppercase;">SUPPLEMENTARY DATA</h2>
                                            <!--/article/back/sec/--><p id="__p55" class="p p-first-last cyxy-trs-source"><a data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Data</a> are available at NAR Online.</p>
                                        </div>
                                        <div id="SEC6" class="tsec bk-sec">
                                            <h2 class="head no_bottom_margin cyxy-trs-source" id="SEC6title" style="text-transform: uppercase;">FUNDING</h2>
                                            <!--/article/back/sec/--><p id="__p56" class="p p-first cyxy-trs-source">Mayo Graduate School and the Mayo Foundation; Mayo Graduate School Harry and Debra Stonecipher Predoctoral Fellowship (to E.M.). The open access publication charge for this paper has been waived by Oxford University Press—<em>NAR</em> Editorial Board members are entitled to one free paper per year in recognition of their work on behalf of the journal.</p>
                                            <p id="__p57" class="p p-last cyxy-trs-source"><em>Conflict of interest statement</em>. None declared.</p>
                                        </div>
                                        <div id="idm140607994728112" class="tsec sec">
                                            <h2 class="head no_bottom_margin cyxy-trs-source" id="idm140607994728112title" style="text-transform: uppercase;">REFERENCES</h2>
                                            <div class="ref-list-sec sec" id="reference-list">
                                                <div class="ref-cit-blk half_rhythm" id="B1">
                                                    1. <span class="element-citation cyxy-trs-source">
                                                        Pingoud A., Jeltsch A. Structure and function of type II restriction endonucleases. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Nucleic Acids Res. </span>2001;<span class="ref-vol">29</span>:3705–3727.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B2">
                                                    2. <span class="element-citation cyxy-trs-source">
                                                        Naito T., Kusano K., Kobayashi I. Selfish behavior of restriction-modification systems. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Science. </span>1995;<span class="ref-vol">267</span>:897–899.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B3">
                                                    3. <span class="element-citation cyxy-trs-source">
                                                        Smith H.O., Kelly T.J., Roy P.H. Enzymatic methods for sequence analysis applied to DNA restriction and methylation sites. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Methods Enzymol. </span>1974;<span class="ref-vol">29</span>:282–294.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B4">
                                                    4. <span class="element-citation cyxy-trs-source">
                                                        Smith H.O., Wilcox K.W. A restriction enzyme from Hemophilus influenzae. I. Purification and general properties. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">J. Mol. Biol. </span>1970;<span class="ref-vol">51</span>:379–391.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B5">
                                                    5. <span class="element-citation cyxy-trs-source">
                                                        Engelke D.R., Ng S.Y., Shastry B.S., Roeder R.G. Specific interaction of a purified transcription factor with an internal control region of 5S RNA genes. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Cell. </span>1980;<span class="ref-vol">19</span>:717–728.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B6">
                                                    6. <span class="element-citation cyxy-trs-source">
                                                        Lee B.M., Xu J., Clarkson B.K., Martinez-Yamout M.A., Dyson H.J., Case D.A., Gottesfeld J.M., Wright P.E. Induced fit and ‘lock and key’ recognition of 5S RNA by zinc fingers of transcription factor IIIA. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">J. Mol. Biol. </span>2006;<span class="ref-vol">357</span>:275–291.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B7">
                                                    7. <span class="element-citation cyxy-trs-source">
                                                        Pelham H.R., Brown D.D. A specific transcription factor that can bind either the 5S RNA gene or 5S RNA. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Proc. Natl. Acad. Sci. U.S.A. </span>1980;<span class="ref-vol">77</span>:4170–4174.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B8">
                                                    8. <span class="element-citation cyxy-trs-source">
                                                        Romby P., Baudin F., Brunel C., Leal&nbsp;de&nbsp;Stevenson I., Westhof E., Romaniuk P.J., Ehresmann C., Ehresmann B. Ribosomal 5S RNA from Xenopus laevis oocytes: conformation and interaction with transcription factor IIIA. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Biochimie. </span>1990;<span class="ref-vol">72</span>:437–452.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B9">
                                                    9. <span class="element-citation cyxy-trs-source">
                                                        Shastry B.S. Transcription factor IIIA (TFIIIA): an update. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Experientia. </span>1993;<span class="ref-vol">49</span>:831–835.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B10">
                                                    10. <span class="element-citation cyxy-trs-source">
                                                        Dubnau J., Struhl G. RNA recognition and translational regulation by a homeodomain protein. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Nature. </span>1996;<span class="ref-vol">379</span>:694–699.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B11">
                                                    11. <span class="element-citation cyxy-trs-source">
                                                        Rivera-Pomar R., Niessing D., Schmidt-Ott U., Gehring W.J., Jackle H. RNA binding and translational suppression by bicoid. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Nature. </span>1996;<span class="ref-vol">379</span>:746–749.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B12">
                                                    12. <span class="element-citation cyxy-trs-source">
                                                        Cassiday L.A., Maher L.J. 3rd. Having it both ways: transcription factors that bind DNA and RNA. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Nucleic Acids Res. </span>2002;<span class="ref-vol">30</span>:4118–4126.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B13">
                                                    13. <span class="element-citation cyxy-trs-source">
                                                        Ellington A.D., Szostak J.W. In vitro selection of RNA molecules that bind specific ligands. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Nature. </span>1990;<span class="ref-vol">346</span>:818–822.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B14">
                                                    14. <span class="element-citation cyxy-trs-source">
                                                        Tuerk C., Gold L. Systematic evolution of ligands by exponential enrichment: RNA ligands to bacteriophage T4 DNA polymerase. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Science. </span>1990;<span class="ref-vol">249</span>:505–510.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B15">
                                                    15. <span class="element-citation cyxy-trs-source">
                                                        Wurster S.E., Maher L.J. 3rd. Selection and characterization of anti-NF-kappaB p65 RNA aptamers. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">RNA. </span>2008;<span class="ref-vol">14</span>:1037–1047.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B16">
                                                    16. <span class="element-citation cyxy-trs-source">
                                                        Cassiday L.A., Maher L.J. 3rd. Yeast genetic selections to optimize RNA decoys for transcription factor NF-kappa B. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Proc. Natl. Acad. Sci. U.S.A. </span>2003;<span class="ref-vol">100</span>:3930–3935.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B17">
                                                    17. <span class="element-citation cyxy-trs-source">
                                                        Lebruska L.L., Maher L.J. 3rd. Selection and characterization of an RNA decoy for transcription factor NF-kappa B. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Biochemistry. </span>1999;<span class="ref-vol">38</span>:3168–3174.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B18">
                                                    18. <span class="element-citation cyxy-trs-source">
                                                        Zhao X., Shi H., Sevilimedu A., Liachko N., Nelson H.C., Lis J.T. An RNA aptamer that interferes with the DNA binding of the HSF transcription activator. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:3755–3761.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B19">
                                                    19. <span class="element-citation cyxy-trs-source">
                                                        Sevilimedu A., Shi H., Lis J.T. TFIIB aptamers inhibit transcription by perturbing PIC formation at distinct stages. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Nucleic Acids Res. </span>2008;<span class="ref-vol">36</span>:3118–3127.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B20">
                                                    20. <span class="element-citation cyxy-trs-source">
                                                        Fukunaga J., Nomura Y., Tanaka Y., Amano R., Tanaka T., Nakamura Y., Kawai G., Sakamoto T., Kozu T. The Runt domain of AML1 (RUNX1) binds a sequence-conserved RNA motif that mimics a DNA element. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">RNA. </span>2013;<span class="ref-vol">19</span>:927–936.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B21">
                                                    21. <span class="element-citation cyxy-trs-source">
                                                        Ghosh G., Huang D.B., Huxford T. Molecular mimicry of the NF-kappaB DNA target site by a selected RNA aptamer. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Curr. Opin. Struct. Biol. </span>2004;<span class="ref-vol">14</span>:21–27.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B22">
                                                    22. <span class="element-citation cyxy-trs-source">
                                                        Huang D.B., Vu D., Cassiday L.A., Zimmerman J.M., Maher L.J. 3rd, Ghosh G. Crystal structure of NF-kappaB (p50)2 complexed to a high-affinity RNA aptamer. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Proc. Natl. Acad. Sci. U.S.A. </span>2003;<span class="ref-vol">100</span>:9268–9273.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B23">
                                                    23. <span class="element-citation cyxy-trs-source">
                                                        Reiter N.J., Maher L.J. 3rd, Butcher S.E. DNA mimicry by a high-affinity anti-NF-kappaB RNA aptamer. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Nucleic Acids Res. </span>2008;<span class="ref-vol">36</span>:1227–1236.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B24">
                                                    24. <span class="element-citation cyxy-trs-source">
                                                        Nomura Y., Tanaka Y., Fukunaga J., Fujiwara K., Chiba M., Iibuchi H., Tanaka T., Nakamura Y., Kawai G., Kozu T., et al.  Solution structure of a DNA mimicking motif of an RNA aptamer against transcription factor AML1 Runt domain. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">J. Biochem. </span>2013;<span class="ref-vol">154</span>:513–519.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B25">
                                                    25. <span class="element-citation cyxy-trs-source">
                                                        Tahirov T.H., Inoue-Bungo T., Morii H., Fujikawa A., Sasaki M., Kimura K., Shiina M., Sato K., Kumasaka T., Yamamoto M., et al.  Structural analyses of DNA recognition by the AML1/Runx-1 Runt domain and its allosteric control by CBFbeta. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Cell. </span>2001;<span class="ref-vol">104</span>:755–767.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B26">
                                                    26. <span class="element-citation cyxy-trs-source">
                                                        Das R., Laederach A., Pearlman S.M., Herschlag D., Altman R.B. SAFA: semi-automated footprinting analysis software for high-throughput quantification of nucleic acid footprinting experiments. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">RNA. </span>2005;<span class="ref-vol">11</span>:344–354.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B27">
                                                    27. <span class="element-citation cyxy-trs-source">
                                                        Soukup G.A., Breaker R.R. Relationship between internucleotide linkage geometry and the stability of RNA. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">RNA. </span>1999;<span class="ref-vol">5</span>:1308–1325.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B28">
                                                    28. <span class="element-citation cyxy-trs-source">
                                                        Oivanen M., Kuusela S., Lonnberg H. Kinetics and mechanisms for the cleavage and isomerization of the phosphodiester bonds of RNA by bronsted acids and bases. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Chem. Rev. </span>1998;<span class="ref-vol">98</span>:961–990.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B29">
                                                    29. <span class="element-citation cyxy-trs-source">
                                                        Zhou D.M., Taira K. The hydrolysis of RNA: from theoretical calculations to the hammerhead ribozyme-mediated cleavage of RNA. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Chem. Rev. </span>1998;<span class="ref-vol">98</span>:991–1026.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B30">
                                                    30. <span class="element-citation cyxy-trs-source">
                                                        SantaLucia J. Jr, Kierzek R., Turner D.H. Effects of GA mismatches on the structure and thermodynamics of RNA internal loops. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Biochemistry. </span>1990;<span class="ref-vol">29</span>:8813–8819.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B31">
                                                    31. <span class="element-citation cyxy-trs-source">
                                                        Brahms J., Mommaerts W.F. A study of conformation of nucleic acids in solution by means of circular dichroism. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">J. Mol. Biol. </span>1964;<span class="ref-vol">10</span>:73–88.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B32">
                                                    32. <span class="element-citation cyxy-trs-source">
                                                        Chauca-Diaz A.M., Jung&nbsp;Choi Y., Resendiz M.J. Biophysical properties and thermal stability of oligonucleotides of RNA containing 7,8-dihydro-8-hydroxyadenosine. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Biopolymers. </span>2015;<span class="ref-vol">103</span>:167–174.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B33">
                                                    33. <span class="element-citation cyxy-trs-source">
                                                        Samejima T., Yang J.T. Optical rotatory dispersion of DNA and RNA. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Biochemistry. </span>1964;<span class="ref-vol">3</span>:613–616.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B34">
                                                    34. <span class="element-citation cyxy-trs-source">
                                                        Saravanan M., Bujnicki J.M., Cymerman I.A., Rao D.N., Nagaraja V. Type II restriction endonuclease R.KpnI is a member of the HNH nuclease superfamily. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:6129–6135.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B35">
                                                    35. <span class="element-citation cyxy-trs-source">
                                                        Wurster S.E., Bida J.P., Her Y.F., Maher L.J. 3rd. Characterization of anti-NF-kappaB RNA aptamer-binding specificity in vitro and in the yeast three-hybrid system. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Nucleic Acids Res. </span>2009;<span class="ref-vol">37</span>:6214–6224.</span>
                                                        <span class="nowrap">[<a class="int-reflink cyxy-trs-source">PMC free article</a>]</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                                <div class="ref-cit-blk half_rhythm" id="B36">
                                                    36. <span class="element-citation cyxy-trs-source">
                                                        Regulski E.E., Breaker R.R. In-line probing analysis of riboswitches. <span class="cyxy-trs-source cyxy-trs-source"><span class="ref-journal cyxy-trs-source">Meth. Mol. Biol. </span>2008;<span class="ref-vol">419</span>:53–67.</span> [<a target="pmc_ext" ref="reftype=pubmed&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord" class="cyxy-trs-source">PubMed</a>] <span class="nowrap">[<a target="pmc_ext" ref="reftype=other&amp;article-id=4551934&amp;issue-id=257108&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" class="cyxy-trs-source">Google Scholar</a>]</span>
                                                    </span>
                                                </div>
                                            </div>
                                        </div>
                                    </div>
                                    <!--post-content--><div class="courtesy-note whole_rhythm small cyxy-trs-source cyxy-trs-source">
                                        <hr>Articles from <span class="acknowledgment-journal-title cyxy-trs-source">Nucleic Acids Research</span> are provided here courtesy of <strong class="cyxy-trs-source">Oxford University Press</strong>
                                    </div>
                                </div>




                            </div>
                            <!-- Book content -->
                        </div>

                        <div id="rightcolumn" class="four_col col last">
                            <!-- Custom content above discovery portlets -->
                            <div class="col6">

                            </div>

                            <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
                                <div class="format-menu">
                                    <h2 class="cyxy-trs-source">Formats:</h2>
                                    <ul>
                                        <li class="selected cyxy-trs-source">Article</li> |
                                        <li>
                                            <a class="cyxy-trs-source">PubReader</a>
                                        </li> |
                                        <li class="epub-link">
                                            <a class="cyxy-trs-source">ePub (beta)</a>
                                        </li> |
                                        <li>
                                            <a class="cyxy-trs-source">PDF (6.7M)</a>
                                        </li> |
                                        <li>
                                            <a data-citationid="PMC4551934" class="citationexporter ctxp cyxy-trs-source">Citation</a>
                                        </li>
                                    </ul>
                                </div>
                            </div>
                            <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons">
                                <h2 class="cyxy-trs-source">Share</h2>
                                <ul>
                                    <li class="facebook">
                                        <a class="cyxy-trs-source">
                                            <img src="" alt="Share on Facebook">
                                            Facebook
                                        </a>
                                    </li>
                                    <li class="twitter">
                                        <a class="cyxy-trs-source">
                                            <img src="" alt="Share on Twitter">
                                            Twitter
                                        </a>
                                    </li>
                                    <li class="gplus">
                                        <a class="cyxy-trs-source">
                                            <img src="" alt="Share on Google Plus">
                                            Google+
                                        </a>
                                    </li>
                                </ul>
                            </div>

                            <div id="ajax-portlets" data-pmid="26184872" data-aiid="4551934" data-aid="4551934" data-iid="257108" data-domainid="4" data-domain="nar" data-accid="PMC4551934" data-md5="36233f128fbf0f260e5f9021937e820a"></div>

                            <!-- Custom content below discovery portlets -->
                            <div class="col7">

                            </div>
                        </div>
                    </div>

                    <!-- Custom content after all -->
                    <div class="col8">

                    </div>
                    <div class="col9">

                    </div>

                    <div id="body-link-poppers"></div>
                </div>
                <div class="bottom">

                    <div id="NCBIFooter_dynamic">
                        <a id="help-desk-link" class="help_desk cyxy-trs-source" target="_blank">Support Center</a>
                        <a id="help-desk-link" class="help_desk cyxy-trs-source" target="_blank">Support Center</a>

                    </div>

                    <div class="footer" id="footer">

                        <div class="subfooter"> </div>
                        <div id="external-disclaimer" class="offscreen_noflow cyxy-trs-source">
                            External link. Please review our <a class="cyxy-trs-source">privacy policy</a>.
                        </div>
                        <div id="ncbifooter" class="contact_info">
                            <div id="footer-contents-right">
                                <div id="nlm_thumb_logo">
                                    <a title="NLM" class="cyxy-trs-source">NLM</a>
                                </div>
                                <div id="nih_thumb_logo">
                                    <a title="NIH" class="cyxy-trs-source">NIH</a>
                                </div>
                                <div id="hhs_thumb_logo">
                                    <a title="DHHS" class="cyxy-trs-source">DHHS</a>
                                </div>
                                <div id="usagov_thumb_logo">
                                    <a title="USA.gov">USA.gov</a>
                                </div>
                            </div>

                            <div id="footer-contents-left">
                                <p class="address vcard cyxy-trs-source">
                                    <span class="url">
                                        <a class="fn url newdomain">
                                            National Center for
                                            Biotechnology Information
                                        </a>,
                                    </span>
                                    <span class="org url newdomain">
                                        <a>U.S. National Library of Medicine</a>
                                    </span>
                                    <span class="adr">
                                        <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                                        <span class="region">MD</span>, <span class="postal-code">20894</span>
                                        <span class="country-name">USA</span>
                                    </span>
                                </p>

                                <a class="cyxy-trs-source">Policies and Guidelines</a> | <a class="cyxy-trs-source">Contact</a>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <!--/.page-->
        </div>
        <!--/.wrap-->
    </div>
    <!-- /.twelve_col -->
</div>
<!-- /.grid -->

<span class="PAFAppResources"></span>

<!-- BESelector tab -->



<noscript>
    <img alt="statistics" src="" />
</noscript>


<!-- usually for JS scripts at page bottom -->
<!--<component id="PageFixtures" label="styles"></component>-->
<!-- 8A1B6232F4540A31_2670SID /projects/PMC/PMCViewer@4.46 ptpmc102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->
